<!DOCTYPE html><html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=2"><meta name="theme-color" content="#FFF"><link rel="alternate" type="application/rss+xml" title="Title" href="https://2021.igem.org/Template:Fudan/Others/rss_xml"><link rel="alternate" type="application/atom+xml" title="Title" href="https://2021.igem.org/Template:Fudan/Others/atom_xml"><link rel="alternate" type="application/json" title="Title" href="https://2021.igem.org/Template:Fudan/Others/feed_json"><link rel="stylesheet" type="text/css" href="https://2021.igem.org/Template:Fudan/CSS/main_css?action=raw&ctype=text/css"><meta name="generator" content="Hexo 5.4.0"></head><body><div id="loading"><img src="https://2021.igem.org/wiki/images/b/b9/T--Fudan--Loading.gif" alt=""></div><div id="container"><header id="header"><div class="inner"><div id="brand"><div class="pjax"><h1 itemprop="name headline">Description</h1></div></div><nav id="nav"><div class="inner"><div class="toggle"><div class="lines" aria-label="Toggle navigation bar"><span class="line"></span> <span class="line"></span> <span class="line"></span></div></div><ul class="menu"><li class="item title"><a target="_blank" href="https://2021.igem.org/Team:Fudan" rel="start noopener">Home</a></li><li class="item dropdown"><a href="javascript:void(0);">Team</a><ul class="submenu"><li class="item"><a href="/archives/" rel="section">Team</a></li><li class="item"><a href="/categories/" rel="section">Attributions</a></li><li class="item"><a href="/tags/" rel="section">Collaborations</a></li></ul></li><li class="item dropdown"><a href="javascript:void(0);">Project</a><ul class="submenu"><li class="item"><a href="/archives/" rel="section">Communication</a></li><li class="item"><a href="/categories/" rel="section">Contribution</a></li><li class="item"><a href="/tags/" rel="section">Description</a></li><li class="item"><a href="/tags/" rel="section">Engineering</a></li><li class="item"><a href="/tags/" rel="section">Experiments</a></li><li class="item"><a href="/tags/" rel="section">Implementation</a></li><li class="item"><a href="/tags/" rel="section">Notebook</a></li><li class="item"><a href="/tags/" rel="section">Partnership</a></li><li class="item"><a href="/tags/" rel="section">Proof Of Concept</a></li><li class="item"><a href="/tags/" rel="section">Results</a></li></ul></li><li class="item dropdown"><a href="javascript:void(0);">Parts</a><ul class="submenu"><li class="item"><a href="/archives/" rel="section">Part Collection</a></li><li class="item"><a href="/categories/" rel="section">Parts</a></li><li class="item"><a href="/tags/" rel="section">Improve</a></li></ul></li><li class="item"><a href="javascript:void(0);">Safety</a></li><li class="item dropdown"><a href="javascript:void(0);">Human Practice</a><ul class="submenu"><li class="item"><a href="/archives/" rel="section">1</a></li><li class="item"><a href="/categories/" rel="section">2</a></li><li class="item"><a href="/tags/" rel="section">3</a></li></ul></li><li class="item dropdown"><a href="javascript:void(0);">Awards</a><ul class="submenu"><li class="item"><a href="/archives/" rel="section">Education</a></li><li class="item"><a href="/categories/" rel="section">Entrepreneurship</a></li><li class="item"><a href="/tags/" rel="section">Hardware</a></li><li class="item"><a href="/tags/" rel="section">Inclusivity</a></li><li class="item"><a href="/tags/" rel="section">Measurement</a></li><li class="item"><a href="/tags/" rel="section">Model</a></li><li class="item"><a href="/tags/" rel="section">Plant</a></li><li class="item"><a href="/tags/" rel="section">Software</a></li><li class="item"><a href="/tags/" rel="section">Sustainable</a></li></ul></li><li class="item"><a href="javascript:void(0);">Poster</a></li></ul><ul class="right"><li class="item theme"><i class="ic i-sun"></i></li></ul></div></nav></div><div id="imgs" class="pjax"><ul><li class="item"><img src="https://tva2.sinaimg.cn/large/6833939bly1giclg5ms2rj20zk0m8u0x.jpg"></li><li class="item"><img src="https://tva2.sinaimg.cn/large/6833939bly1gipexj2jgzj20zk0m8b09.jpg"></li><li class="item"><img src="https://tva2.sinaimg.cn/large/6833939bly1gipesx5fdwj20zk0m81kx.jpg"></li><li class="item"><img src="https://tva2.sinaimg.cn/large/6833939bly1gicitzannuj20zk0m8b29.jpg"></li><li class="item"><img src="https://tva2.sinaimg.cn/large/6833939bly1giclip4jbpj20zk0m87cv.jpg"></li><li class="item"><img src="https://tva2.sinaimg.cn/large/6833939bly1gicliierfjj20zk0m8npd.jpg"></li></ul></div></header><div id="waves"><svg class="waves" viewBox="0 24 150 28" preserveAspectRatio="none" shape-rendering="auto"><defs><path id="gentle-wave" d="M-160 44c30 0 58-18 88-18s 58 18 88 18 58-18 88-18 58 18 88 18 v44h-352z"/></defs><g class="parallax"><use xlink:href="#gentle-wave" x="48" y="0"/><use xlink:href="#gentle-wave" x="48" y="3"/><use xlink:href="#gentle-wave" x="48" y="5"/><use xlink:href="#gentle-wave" x="48" y="7"/></g></svg></div><main><div class="inner"><div id="main" class="pjax"><div class="page wrap"><article itemscope itemtype="" class="post block" lang="en"><div class="body md" itemprop="articleBody"><h1 id="inspiration-and-original-design"><a class="anchor" href="#inspiration-and-original-design">#</a> Inspiration and Original Design</h1><p>In May 2020, Fudan University launched <em>Hands Out Free Sanitary Pads in Schools Campaign</em>- women's restrooms in each teaching building are equipped with a sanitary towel mutual aid bag containing one or two separately packaged sanitary napkins to provide women who forget to bring sanitary napkins to ease the embarrassment. Fudan was not the only university in Shanghai to do so, The intention of the event was good, but it also made us think: <strong>Will the prolonged placement of sanitary napkins in exposed, moist environments lead to safety problems such as mold breeding?</strong></p><p><img data-src="https://2021.igem.org/wiki/images/9/97/T--Fudan--description2.jpg" alt="pic"><br>Fig.1 Sanitary napkins in the women restroom</p><p>At first our project intended to focus on the detection and safety of sanitary napkins, and to design a sticker that would change color based on the number of bacteria and mold on the surface to indicate whether sanitary napkins could be used. <strong>Our original project design was to detect germs on the surface of sanitary napkins based on group sensing and cascade the signal with a circuit of synthetic biology.</strong></p><p><img data-src="https://2021.igem.org/wiki/images/c/c7/T--Fudan--description1.jpg" alt="pic"><br>Fig.2 Our original design of the gene circuits detecting pathogens on sanitary napkins</p><h1 id="detection-of-candida-albicans"><a class="anchor" href="#detection-of-candida-albicans">#</a> Detection of <em>Candida Albicans</em></h1><p>After communicating with our instructor, however, we realized that the original project design had high detection limits and high costs and that the practical value was not great. <strong>We turned our attention from the sanitary napkin to the user's health</strong>, and after data search, we found that the original vaginal disease causes are very many, the main category is Vulvovaginal Candidiasis after the team members asked us to understand that the original prevalence of vaginal disease far exceeded our expectations, for the detection of the disease is fully necessary.</p><p><img data-src="https://2021.igem.org/wiki/images/6/65/T--Fudan--Candida.png" alt="pic"><br>Fig3. Picture of <em>C. albicans</em></p><p>Candidiasis is one of the most common opportunistic fungal infections which affects besides vaginal the skin, nails, bronchial, lungs, and digestive mucosal surfaces acutely, subacute, and chronically (Mirhandi et al. 2006). Sometimes, the disseminated infection may also occur in the kidneys, liver, and heart (Mirhandi et al. 2006; Chadwick et al. 2013). In immunocompromised patients, the <em>Candida</em> species are among the main causes of sepsis, and nosocomial bloodstream infections which have been associated with significant mortality (Lion 2017). Also, <em>Candida</em> species can cause infections of the prosthesis, the urinary tract, and the upper respiratory tract. <em>C. albicans</em> has an indisputable ability to develop human infections, as is the most common fungal infectious agent in humans, the most common fungal agent of hospital infections, and the fourth cause of nosocomial bloodstream infections among all microbial agents that cause infection (Coronado-Castellote and Jiménez-Soriano 2013; Sherry et al. 2014).</p><p>Therefore, accurate detection of <em>Candida</em> species has a significant role in the prevention, control, and timely treatment of the disease. Our project focuses on the most common disease caused by <em>Candida albicans</em> among women but may also be applied out of VVC to a broader range of <em>Candida</em> species-caused diseases.</p><h1 id="current-detection-methods-and-their-weaknesses"><a class="anchor" href="#current-detection-methods-and-their-weaknesses">#</a> Current detection methods and their weaknesses</h1><p>The routine diagnosis of <em>Candida albicans</em> in the laboratory is carried out using diagnostic methods such as CHROMagar <em>Candida</em>, germ tube production, chlamydoconidia generation, and carbohydrate assimilation assays. The CHROMagar <em>Candida</em> medium is cost-effective but requires much time to achieve the results, and this method cannot differentiate all <em>Candida</em> species. The germ tube production method is not accurate and has false-negative results. Molecular detection methods based on the isolation and amplification of nucleic acids of the fungus include a variety of polymerase chain reaction (PCR) assays, hybridization, and isothermal amplification technique, which can detect the infection even in the primary stages of the disease. The PCR assay is a fast and reliable method that has not only many advantages but also some limitations including various temperature cycles, the use of expensive thermocyclers and gel documentation systems, and variable sensitivity and specificity. Therefore, this method cannot be used in low-income countries or resource-limited districts.[1]</p><h1 id="the-goal-of-our-project"><a class="anchor" href="#the-goal-of-our-project">#</a> The goal of our project</h1><p>Since the rapid and accurate identification of <em>C. albicans</em> from the non-<em>albicans</em> species is essential, especially in people who have a recurrent infection, <strong>there's an urgent need to discover cost-effective and user-friendly methods with high sensitivity and specificity to detect <em>C. albicans</em></strong>, which is also the goal of our project. Above is the origin of our project.</p><p>We were not the only team to focus on the detection of <em>C. albicans</em>, <span class="exturl" data-url="aHR0cDovLzIwMTguaWdlbS5vcmcvVGVhbTpVaU9zbG9fTm9yd2F5">Team UiOslo_Norway</span> in 2018 also focused on the detection of <em>C. albicans</em> using CRISPR/dCas9. While they focused on the accuracy and detection limits, we care about lowering the cost of detection by improving the production of the enzyme entailed in the detection.</p><p>[1] Elie, C. M., Lott, T. J., Reiss, E., &amp; Morrison, C. J. (1998). Rapid identification of Candida species with species-specific DNA probes. <em>Journal of clinical microbiology</em>, <em>36</em>(11), 3260–3265. <span class="exturl" data-url="aHR0cHM6Ly9kb2kub3JnLzEwLjExMjgvSkNNLjM2LjExLjMyNjAtMzI2NS4xOTk4">https://doi.org/10.1128/JCM.36.11.3260-3265.1998</span></p><h1 id="overview"><a class="anchor" href="#overview">#</a> <strong>Overview</strong></h1><p>As a conditionally pathogenic strain, Candida albicans can colonize on genital/gastrointestinal mucosa without causing disease, and only in immunosuppressed hosts can C. Albicans become pathogenic. Vulva vaginal inflammation caused by candida infection is one of the most common vulva inflammation in women. [1] However, the existing detection methods still have problems such as low accuracy in clinical testing, high cost, and long cycles in laboratory testing. So we tried to build a product for the rapid, cheap, and accurate detection of Candida albicans, which can be used in resource-limited regions as well.</p><p>**We used E.coli to over-express the Bst DNA polymerase entailed in loop-mediated isothermal amplification (LAMP) reaction.**Such an enzyme amplifies the specific DNA sequence of Candida albicans to achieve the purpose of rapid and sensitive detection. Besides, <strong>we designed gene circuits with our core parts gp2 and gp5.7 to suppress the background protein translation and uplift the relative purity of Bst polymerase produced by E.coli, and by fusing Bst with various kinds of DNA binding proteins, we are hoping to reduce the cost of the detection and increase the efficiency simultaneously.</strong></p><h1 id="isothermal-amplification"><a class="anchor" href="#isothermal-amplification">#</a> Isothermal amplification</h1><p>The routine diagnosis of <em>Candida</em>albicans in the laboratory is carried out using diagnostic methods such as CHROMagar <em>Candida</em>, germ tube production, chlamydoconidia generation, and carbohydrate assimilation assays. The CHROMagar Candida medium is cost effective but requires much time to achieve the results, and this method cannot differentiate all Candida species. The germ tube production method is not accurate and has false-negative results. Molecular detection methods based on the isolation and amplification of nucleic acids of the fungus include a variety of polymerase chain reaction (PCR) assays, hybridization, and isothermal amplification technique, which can detect the infection even in the primary stages of the disease. The PCR assay is a fast and reliable method that has not only many advantages but also some limitations including various temperature cycles, the use of expensive thermocyclers and gel documentation system, and variable sensitivity and specificity. Therefore, this method cannot be used in low income countries or in resource limited districts.[2]</p><p>We turned to isothermal ammplification to amplifay the target DNA of C.albicans. Isothermal amplifification of nucleic acids is a simple process that rapidly</p><p>and efficiently accumulates nucleic acid sequences at constant temperature.</p><h2 id="loop-mediated-isothermal-amplification-lamp"><a class="anchor" href="#loop-mediated-isothermal-amplification-lamp">#</a> Loop-mediated isothermal amplification (LAMP)</h2><h3 id="why-did-we-choose-lamp"><a class="anchor" href="#why-did-we-choose-lamp">#</a> Why did we choose LAMP?</h3><p>With the aim to create a rapid, specific and cost-effective point-of-care detection system, at first, we needed to find the most suitable isothermal DNA amplification method. This method should be usable for farmers who have no scientific background. This factor pinpoints a huge need to be able to perform these isothermal reactions with as minimal pipetting steps as possible by means of avoiding errors and false-positive results. Although, amplification of marker sequences should be done in constant temperature by the needs of cheap and fully-portable equipment.</p><p>Considering these main requirements, we have distinguished some isothermal amplification methods and concluded that loop-mediated isothermal amplification technique (LAMP) should be the solution for the following reasons. First, The isothermal amplifification of target species-specifific DNA sequence through LAMP does not require thermocycler thus offers potential advantages of speed, cost, simplicity, and accuracy.[4] Second, LAMP method is highly sequencing specifific, high amplifification specifificity allows the synthesis of a large amount of DNA in a very short time, the detection limit is few copies per reaction, and requires only a heating block or a water bath at a constant temperature [5]. Last but not least, LAMP requires simple enzyme as the Bst DNA polymerase which we intended to improve using sythetic biology methods.</p><p>~<br>Comparison between different isothermal amplification methods[3]</p><p>However, the LAMP method has few limitations, such as LAMP amplifified DNA products can be visualized by gel electrophoresis or other detection methods. However, these methods may require the opening of the reaction tubes and increase the risk of carry-over contamination and complexity. A onepot reaction is helpful and the end-product can be visualized with</p><p>the naked eye, but a nakedeye method is not sensitive to variability among various samples and the detection limit is very high and the specificity can't be ensured. We prefered to use the lateral flow asssay to visualize the results as descreibed <span class="exturl" data-url="aHR0cDovL2JlbG93Lmlu">below.in</span> Lateral flow assay (LFA).</p><h3 id="choosing-marker-gene-and-design-of-primers-and-probes"><a class="anchor" href="#choosing-marker-gene-and-design-of-primers-and-probes">#</a> Choosing marker gene and design of primers and probes</h3><p>After bioinformatic analysis of the genome DNA using BLAST and referencing existing PCR detection method of <em>C.albicans</em> [6], we designed to target the ITS2 region of the 5.8S rDNA of <em>C.albicans</em>, and the primers and probe are as follow:</p><p>The LAMP primer sets were designed using Primer Explorer Ver. 4 (<span class="exturl" data-url="aHR0cDovL3ByaW1lcmV4cGxvcmVyLmpwL2Uv">http://primerexplorer.jp/e/</span>). For Lateral Flow Assay (LF) Assay detection, one detection probe was tagged with biotin at the 50 end, which has been entailed in th primer, and the other with FITC at 30 end. The newly designed probes were screened using the oligoanalyzer and BLAST tool (<span class="exturl" data-url="aHR0cDovL3d3dy5uY2JpLm5sbS5uaWgu">http://www.ncbi.nlm.nih.</span> gov/BLAST).</p><table><thead><tr><th style="text-align:left">Name</th><th style="text-align:left">Sequence</th><th style="text-align:left">type</th></tr></thead><tbody><tr><td style="text-align:left">F3</td><td style="text-align:left">TGGGTTTGCTTGAAAGACGG</td><td style="text-align:left">primer</td></tr><tr><td style="text-align:left">B3</td><td style="text-align:left">CTTCACTCGCCGCTACTG</td><td style="text-align:left"></td></tr><tr><td style="text-align:left">FIP</td><td style="text-align:left">CCGCCGCAAGCAATGTTTTTGGTAGTGGTAAGGCGGGATCG</td><td style="text-align:left"></td></tr><tr><td style="text-align:left">biotin-BIP</td><td style="text-align:left">ATCAGGTAGGACTACCCGCTGAAGGCAATCCCTGTTGGTTTC</td><td style="text-align:left"></td></tr><tr><td style="text-align:left">LF</td><td style="text-align:left">TTGACAATGGCTTAGGTCTA</td><td style="text-align:left"></td></tr><tr><td style="text-align:left">LB</td><td style="text-align:left">GCATATCAATAAGCGGAGGA</td><td style="text-align:left"></td></tr><tr><td style="text-align:left">FITC-probe</td><td style="text-align:left">ATTGCTTGCGGCGGTAACGTCC</td><td style="text-align:left">probe</td></tr></tbody></table><h2 id="lateral-flow-assaylfa"><a class="anchor" href="#lateral-flow-assaylfa">#</a> Lateral flow assay（LFA）</h2><p>The lateral flow immunoassay (LFA) is an immune chromatographic technique conducted on a lateral flow dipstick (LFD). LFA has been employed successfully to achieve point-of-care diagnostics [7]. LFA is specifific and sensitive, does not require a specifific instrument. Presently, rapid, sensitive, specifific, and affordable point-of-care tests are needed, especially in places that do not have access to laboratory services.<br>~<br>Schematic of the detection principle of LAMP-LFA</p><p>Amplicon specific self-designed tagged probes further confirmed species-specifific LAMP amplifification and ruled out misinterpretation due to any spurious amplifification. We demonstrated the simplicity and feasibility of this assay by detecting DNA extracted from Candida albicans and we also tested this assay under various backgrounds and processing conditions and high stability and sensitivity were obtained. From sampling to the detection of end-product was completed in less than 180 min and the only device that is needed in resource limited area is a thermos so that the detection can be point-of -care.</p><h1 id="applying-synthetic-biologydesigning-a-brand-new-t7-phage-expressing-system-to-lowering-the-cost-of-the-detection"><a class="anchor" href="#applying-synthetic-biologydesigning-a-brand-new-t7-phage-expressing-system-to-lowering-the-cost-of-the-detection">#</a> <strong>Applying synthetic biology：Designing a brand new T7 phage expressing system to lowering the cost of the detection</strong></h1><p></p><h2 id="a-new-idea-to-improve-the-enzyme-manufacturing-procedure"><a class="anchor" href="#a-new-idea-to-improve-the-enzyme-manufacturing-procedure">#</a> <strong>A new idea to improve the enzyme manufacturing procedure</strong></h2><p>In ordinary protein manufacturing procedure, we usually pay most attention to protein’s amount and activity, and one of the most common ways to achieve both high quantity and quality is linking tags to protein of interest and have it purified.</p><p></p><p>Purification of protein costs much, and it also leads to the problem of ‘how to transport and store these protein’. However, in our project, the target users are mainly those suffering poverty or living in underdeveloped regions. They may have no access to institution with lab environment, or just can’t afford the high cost. To cut these expense, we decide to use the extract of our engineered E.coli to directly catalyze the LAMP reaction. However, there’ll be lots of unrelated proteins in the extract disturbing the reaction, so we need our engineered E.coli to produce our protein of interest, the enzyme, as pure as possible. In another word, we need to raise the relative amount of our protein of interest.</p><p></p><p>After careful consideration, we decide to realize our idea by inhibiting the expression of all E.coli’s proteins except the protein of interest.</p><p></p><h2 id="ecoli-rnap-inhibitor"><a class="anchor" href="#ecoli-rnap-inhibitor">#</a> <strong>E.coli RNAP inhibitor</strong></h2><h3 id="background"><a class="anchor" href="#background">#</a> <strong>Background</strong></h3><p>During the transcription process of E.coli, transcription factors called ‘σ factors (sigma factors)’ are essential for guiding the RNA Polymerase (RNAP) to the promoters and initiating the transcription. We call RNAPs combined with σ factors ‘RNAP whole enzyme’. Each kinds of σ factors can specifically recognize corresponding promoters, thus realizing the overall regulation of gene expression at transcription level. There are at least seven kinds of σ factors that respectively be responsible to dominate the whole transcription process of host E.coli at different situations, including regular growth stages and extreme hazard. Among them, σ70 and σS are two major σ factors of E.coli. According to Abhishek Mazumder and Achillefs N. Kapanidis (2019) and Herb E. Schellhorn (2020), σ70 is the so-called ‘housekeeping’ σ factor which controls about 80% genes. In another word, when the E.coli is in ordinary environment, σ70 will always be the dominant σ factor. On the other hand, σS is the σ factor used by E.coli in order to counter starvation situation. In lab situation, we typically divide bacteria’s growth into four stages. According to the features of σ70 and σS, σS will take dominance at stationary stage while σ70 will show most significance at exponential stage. However, RNAP of phages, such as T7 phage’s RNAP, doesn’t necessarily rely on σ factors.</p><p></p><p>So it suddenly dawned on us that we can interfere σ factors to inhibit transcription and use phage’s transcription system to make our protein of interest immune to the inhibition. To interfere σ factors, we think of two ways. The first one is to let the engineered E.coli express dominant negative mutant of σ factor. But there’re so many kinds of σ factors, and their conserved regions are quite complicated. It’s hard to design an appropriate dominant negative mutant. Meanwhile, σ factors are quite huge as proteins, overexpress them is contradictory to our aim of reducing irrelevant proteins. The second way is to hinder the ability of σ factors. Then what we need to do is to find a proper inhibitor.</p><p></p><h3 id="inspired-by-mechanism-of-phages"><a class="anchor" href="#inspired-by-mechanism-of-phages">#</a> <strong>Inspired by mechanism of phages</strong></h3><p>Phage, well-known for being the virus infecting bacteria, is introduced in almost all biology text books in middle school. Briefly as the introduction is, they must tell you one phage’s key feature,’ Phages have the ability to shut bacteria down and turn them into factories that help phages replicate.’</p><p></p><p>In the field of synthesis biology, expressing systems of phages are good tools to help you achieve high protein yield. Among them, T7 system is one of the most mature expressing systems. However, the typical T7 expressing system merely utilizing T7 phage’s RNA polymerase, in another word, only materializing phage’s ability to turn bacteria into ‘factories’. Actually, there’s also huge potential in putting phage’s ability of ‘shutting down’ into expressing system.</p><p></p><p>One of T7’s shutting-down-methods is to inhibit the transcription initialization of bacteria by producing three proteins, respectively called gp2, gp5.7 and gp0.7. This process can redistribute the metabolism pressure of the host bacteria, letting them save mass and energy to produce what T7 phages want them to produce, meanwhile, reducing the relevant amount of other proteins for sure. That’s exactly the effects we want.</p><p></p><p>Among the three transcription inhibitors mentioned above, gp2 and gp5.7 are both binding proteins of RNAP whole-enzyme, while gp0.7 is a kinase with low specificity. In order to ensure the system’s robustness, we select gp2 and gp5.7 as candidates for constructing our gene circuit.</p><p></p><h3 id="gp2"><a class="anchor" href="#gp2">#</a> <strong>GP2</strong></h3><p>GP2 accounts for ‘T7 Gene Product 2’. It’s an gene product produced by E.coli infected by T7 phage at its early stage. According to Brian Bae et al.(2013), GP2 takes effects by simultaneously binding to RNAP β’ subunit as well as σ70 1.1domain, thus blocking the way DNA should be able to get in. By this process, all transcription process initiated by σ70 can be inhibited.</p><p>~</p><p>We are not the first team to propose GP2 in iGem community. In iGem 2018, Team: Imperial used T7 GP2 to hinder E.coli from growing without killing them, but they didn’t explain why GP2 can’t kill E.coli. We propose two hypothesis about this question. The first one is that, GP2 can inhibit the transcription of the mRNA of itself, thus reaching a balance. (这里最好想要一个 modeling 的图) The second is that, σS will take over σ70 after GP2 is expressed, and keep the cell surviving. Therefore, it’s necessary to introduce a σS-RNAP inhibitor to ensure the inhibition effect.</p><p></p><h3 id="gp57"><a class="anchor" href="#gp57">#</a> <strong>GP5.7</strong></h3><p>GP5.7 accounts for ‘T7 Gene Product 5.7’. It’s an gene product produced by E.coli infected by T7 phage at its middle stage. According to Aline Tabib-Salazar et al.(2018), GP5.7 takes effects by simultaneously binding to RNAP β’ subunit as well as σS R4 domain. By this process, all transcription process initiated by σS can be inhibited.</p><h3 id="σs-promoters"><a class="anchor" href="#σs-promoters">#</a> <strong>σS promoters</strong></h3><p>To prove the effect of GP2 and GP5.7. We need several σ70 promoters and σ70 promoters. While σ70 promoters are everywhere, σS promoters are super rare in iGem community. To solve this problem, we decide to introduce a set of σS promoters ourselves.</p><p>According to Regine Hengge-Aronis(2002), the regulation of σS doesn’t only includes the expression of σS. There’s a complicated regulation procedure. According to Anna Macia˛ g et al.(2011), σS promoters usually feature several UP elements that enable other factors to regulate them. In order to minimize the irrelevant factors, we only cut out about 35bp from those whole σS promoters. Since that might make the promoter lose its properties, we design an experiment to test and verify all the simplified σS promoters we propose.</p><p>σS promoters we propose include: CsiD, osmY, fic and dps, the sequences of which are shown below.</p><p>~<br>Gisela Becker and Regine Hengge-Aronis(2001) has analyzes several σS promoters sequence and found that they have quite conserved sequences in their simplest form. Since csiD and fic didn’t show as strong evidence of being σS promoters as dps and osmY, we decide to improve their sequence manually by introducing spot mutation following Gisela Becker and Regine Hengge-Aronis’s results. The sequence of improved promoters are shown below.</p><p>~</p><h1 id="designing-fusion-proteins-to-enhance-bst-dna-polymerase-activity"><a class="anchor" href="#designing-fusion-proteins-to-enhance-bst-dna-polymerase-activity">#</a> <strong>Designing fusion proteins to enhance Bst DNA polymerase activity</strong></h1><h2 id="about-the-double-stranded-binding-protein-sso7d"><a class="anchor" href="#about-the-double-stranded-binding-protein-sso7d">#</a> <strong>About the double-stranded binding protein Sso7d</strong></h2><p>Sso7d is a kind of double-stranded binding protein, and it be used widely to improve the activity of polymerase[[](file:///C:/Users/25411/Desktop/%E6%96%B0%E5%BB%BA%20DOCX%20%E6%96%87%E6%A1%A3%20(3).docx#_edn1)[7](file:///C:/Users/25411/Desktop/%E6%96%B0%E5%BB%BA%20DOCX%20%E6%96%87%E6%A1%A3%20(3).docx#_edn1)[]](file:///C:/Users/25411/Desktop/%E6%96%B0%E5%BB%BA%20DOCX%20%E6%96%87%E6%A1%A3%20(3).docx#_edn1). The DNA structure changes at different concentrations of Sso7d and depends on the reaction time. At relatively low concentrations of Sso7d, DNA strands form a kinked structure. When the concentration of Sso7d increases, DNA loops appear. Finally, DNA becomes a dense nuclear structure at high concentrations of Sso7d. If the time of interaction between Sso7d and DNA increases, the DNA structure tends to be more compact. High concentrations of Sso7d are important for the compact structure of DNA[[](file:///C:/Users/25411/Desktop/%E6%96%B0%E5%BB%BA%20DOCX%20%E6%96%87%E6%A1%A3%20(3).docx#_edn2)[8](file:///C:/Users/25411/Desktop/%E6%96%B0%E5%BB%BA%20DOCX%20%E6%96%87%E6%A1%A3%20(3).docx#_edn2)[]](file:///C:/Users/25411/Desktop/%E6%96%B0%E5%BB%BA%20DOCX%20%E6%96%87%E6%A1%A3%20(3).docx#_edn2). Therefore, Sso7d has an affinity for double-stranded binding proteins and thus recognizes the annealed double-strand, allowing the polymerase to quickly find the extension site, thus effectively increasing the extension speed of the DNA polymerase.</p><p></p><h2 id="a-new-idea-to-enhance-the-activity-of-bst-dna-polymerase"><a class="anchor" href="#a-new-idea-to-enhance-the-activity-of-bst-dna-polymerase">#</a> <strong>A new idea to enhance the activity of Bst DNA polymerase</strong></h2><p>It has been widely reported in the past that the activity associated with Pfu DNA polymerase can be greatly improved by adding the double-stranded binding protein Sso7d1 to the C-terminus of Pfu DNA polymerase. Meanwhile, our laboratory has done experiments to improve the activity of DNA polymerase Ⅱ by adding double-stranded binding protein in the past and has accumulated more experience[[](file:///C:/Users/25411/Desktop/%E6%96%B0%E5%BB%BA%20DOCX%20%E6%96%87%E6%A1%A3%20(3).docx#_edn3)[9](file:///C:/Users/25411/Desktop/%E6%96%B0%E5%BB%BA%20DOCX%20%E6%96%87%E6%A1%A3%20(3).docx#_edn3)[]](file:///C:/Users/25411/Desktop/%E6%96%B0%E5%BB%BA%20DOCX%20%E6%96%87%E6%A1%A3%20(3).docx#_edn3). In previous studies, Sso7d was suggested to synthesize fusion proteins with DNA polymerase A or DNA polymerase B to largely increase polymerase activity, but no studies have been conducted on the changes in the activity of DNA polymerase Ⅰ and Sso7d fusion proteins. Even so, we still speculated that the activity of Bst DNA polymerase might be improved by linking double-stranded binding proteins, and selected four double-stranded binding proteins from the same hyperthermophilic archaeon Sulfolobus solfataricus, Sso7d, Sso10b, DbpA and albA1, for our experiments. Based on the results of previous modifications of pfu DNA polymerase, we predicted that the bst-sso7d fusion protein might possess the highest activity.</p><p></p><h2 id="our-selection-of-binding-proteins"><a class="anchor" href="#our-selection-of-binding-proteins">#</a> <strong>Our selection of binding proteins</strong></h2><p>We selected six binding proteins for our experiments ( two for single-stranded binding protein and four for double-stranded binding protein). We asseumed that just as double-stranded binding proteins can bind annealed double-stranded DNA, single-stranded binding proteins can also bind denatured single-stranded DNA and thus improve the amplification efficiency of DNA polymerase. In addition to this, we designed two possible twelve-component experiments by ligating the binding proteins to the N-terminal and C-terminal ends of Bst DNA polymerase, respectively, for the experiments. In designing the fusion protein, we chose the flexible linker ((G2S)3) as the linker of our fusion protein.</p><h3 id="according-to-the-consequence-of-our-experiments-our-modification-actually-works-as-we-expected"><a class="anchor" href="#according-to-the-consequence-of-our-experiments-our-modification-actually-works-as-we-expected">#</a> <strong>According to the consequence of our experiments, our modification actually works as we expected.</strong></h3><h2 id="references"><a class="anchor" href="#references">#</a> References</h2><p>[1] Poulain D. (2015). Candida albicans, plasticity and pathogenesis. <em>Critical reviews in microbiology</em>, <em>41</em>(2), 208–217. <span class="exturl" data-url="aHR0cHM6Ly9kb2kub3JnLzEwLjMxMDkvMTA0MDg0MVguMjAxMy44MTM5MDQ=">https://doi.org/10.3109/1040841X.2013.813904</span></p><p>[2] Elie, C. M., Lott, T. J., Reiss, E., &amp; Morrison, C. J. (1998). Rapid identification of Candida species with species-specific DNA probes. <em>Journal of clinical microbiology</em>, <em>36</em>(11), 3260–3265. <span class="exturl" data-url="aHR0cHM6Ly9kb2kub3JnLzEwLjExMjgvSkNNLjM2LjExLjMyNjAtMzI2NS4xOTk4">https://doi.org/10.1128/JCM.36.11.3260-3265.1998</span></p><p>[3] Zhao, Y., Chen, F., Li, Q., Wang, L., &amp; Fan, C. (2015). Isothermal Amplification of Nucleic Acids. <em>Chemical reviews</em>, 115(22), 12491–12545. <span class="exturl" data-url="aHR0cHM6Ly9kb2kub3JnLzEwLjEwMjEvYWNzLmNoZW1yZXYuNWIwMDQyOA==">https://doi.org/10.1021/acs.chemrev.5b00428</span></p><p>[4] Wong, Y. P., Othman, S., Lau, Y. L., Radu, S., &amp; Chee, H. Y. (2018). Loop-mediated isothermal amplification (LAMP): a versatile technique for detection of micro-organisms. <em>Journal of applied microbiology</em>, <em>124</em>(3), 626–643. <span class="exturl" data-url="aHR0cHM6Ly9kb2kub3JnLzEwLjExMTEvamFtLjEzNjQ3">https://doi.org/10.1111/jam.13647</span></p><p>[5] Fallahi, S., Babaei, M., Rostami, A. <em>et al.</em> Diagnosis of <em>Candida albicans</em>: conventional diagnostic methods compared to the loop-mediated isothermal amplification (LAMP) assay. <em>Arch Microbiol</em> **202,**275–282 (2020). <span class="exturl" data-url="aHR0cHM6Ly9kb2kub3JnLzEwLjEwMDcvczAwMjAzLTAxOS0wMTczNi03">https://doi.org/10.1007/s00203-019-01736-7</span></p><p>[6] Koczula, K. M., &amp; Gallotta, A. (2016). Lateral flow assays. <em>Essays in biochemistry</em>, <em>60</em>(1), 111–120. <span class="exturl" data-url="aHR0cHM6Ly9kb2kub3JnLzEwLjEwNDIvRUJDMjAxNTAwMTI=">https://doi.org/10.1042/EBC20150012</span></p><p>[[7]](file:///C:/Users/25411/Desktop/%E6%96%B0%E5%BB%BA%20DOCX%20%E6%96%87%E6%A1%A3%20(3).docx#_ednref1) Wang Y, Prosen DE, Mei L, Sullivan JC, Finney M, Vander Horn PB. A novel strategy to engineer DNA polymerases for enhanced processivity and improved performance in vitro. Nucleic Acids Res. 2004 Feb 18;32(3):1197-207. doi: 10.1093/nar/gkh271. PMID: 14973201; PMCID: PMC373405.</p><p>[[8]](file:///C:/Users/25411/Desktop/%E6%96%B0%E5%BB%BA%20DOCX%20%E6%96%87%E6%A1%A3%20(3).docx#_ednref2) Kalichuk, V., Béhar, G., Renodon-Cornière, A. et al. The archaeal “7 kDa DNA-binding” proteins: extended characterization of an old gifted family. Sci Rep 6, 37274 (2016). <span class="exturl" data-url="aHR0cHM6Ly9kb2kub3JnLzEwLjEwMzgvc3JlcDM3Mjc0">https://doi.org/10.1038/srep37274</span></p><p>[[9]](file:///C:/Users/25411/Desktop/%E6%96%B0%E5%BB%BA%20DOCX%20%E6%96%87%E6%A1%A3%20(3).docx#_ednref3) Cao S-C, Qiu L-Z. Study of DNA binding protein DbpA affecting the performance of DNA polymerase[J]. Journal of Fudan:Natural Science Edition, 2015, 54(4):469-477.</p><h1 id="overview-2"><a class="anchor" href="#overview-2">#</a> <strong>Overview</strong></h1><p>As a conditionally pathogenic strain, Candida albicans can colonize on genital/gastrointestinal mucosa without causing disease, and only in immunosuppressed hosts can C. Albicans become pathogenic. Vulva vaginal inflammation caused by candida infection is one of the most common vulva inflammation in women. [1] However, the existing detection methods still have problems such as low accuracy in clinical testing, high cost, and long cycles in laboratory testing. So we tried to build a product for the rapid, cheap, and accurate detection of Candida albicans, which can be used in resource-limited regions as well.</p><p>**We used E.coli to over-express the Bst DNA polymerase entailed in loop-mediated isothermal amplification (LAMP) reaction.**Such an enzyme amplifies the specific DNA sequence of Candida albicans to achieve the purpose of rapid and sensitive detection. Besides, <strong>we designed gene circuits with our core parts gp2 and gp5.7 to suppress the background protein translation and uplift the relative purity of Bst polymerase produced by E.coli, and by fusing Bst with various kinds of DNA binding proteins, we are hoping to reduce the cost of the detection and increase the efficiency simultaneously.</strong></p><h1 id="isothermal-amplification-2"><a class="anchor" href="#isothermal-amplification-2">#</a> Isothermal amplification</h1><p>The routine diagnosis of <em>Candida</em>albicans in the laboratory is carried out using diagnostic methods such as CHROMagar <em>Candida</em>, germ tube production, chlamydoconidia generation, and carbohydrate assimilation assays. The CHROMagar Candida medium is cost effective but requires much time to achieve the results, and this method cannot differentiate all Candida species. The germ tube production method is not accurate and has false-negative results. Molecular detection methods based on the isolation and amplification of nucleic acids of the fungus include a variety of polymerase chain reaction (PCR) assays, hybridization, and isothermal amplification technique, which can detect the infection even in the primary stages of the disease. The PCR assay is a fast and reliable method that has not only many advantages but also some limitations including various temperature cycles, the use of expensive thermocyclers and gel documentation system, and variable sensitivity and specificity. Therefore, this method cannot be used in low income countries or in resource limited districts.[2]</p><p>We turned to isothermal ammplification to amplifay the target DNA of C.albicans. Isothermal amplifification of nucleic acids is a simple process that rapidly</p><p>and efficiently accumulates nucleic acid sequences at constant temperature.</p><h2 id="loop-mediated-isothermal-amplification-lamp-2"><a class="anchor" href="#loop-mediated-isothermal-amplification-lamp-2">#</a> Loop-mediated isothermal amplification (LAMP)</h2><h3 id="why-did-we-choose-lamp-2"><a class="anchor" href="#why-did-we-choose-lamp-2">#</a> Why did we choose LAMP?</h3><p>With the aim to create a rapid, specific and cost-effective point-of-care detection system, at first, we needed to find the most suitable isothermal DNA amplification method. This method should be usable for farmers who have no scientific background. This factor pinpoints a huge need to be able to perform these isothermal reactions with as minimal pipetting steps as possible by means of avoiding errors and false-positive results. Although, amplification of marker sequences should be done in constant temperature by the needs of cheap and fully-portable equipment.</p><p>Considering these main requirements, we have distinguished some isothermal amplification methods and concluded that loop-mediated isothermal amplification technique (LAMP) should be the solution for the following reasons. First, The isothermal amplifification of target species-specifific DNA sequence through LAMP does not require thermocycler thus offers potential advantages of speed, cost, simplicity, and accuracy.[4] Second, LAMP method is highly sequencing specifific, high amplifification specifificity allows the synthesis of a large amount of DNA in a very short time, the detection limit is few copies per reaction, and requires only a heating block or a water bath at a constant temperature [5]. Last but not least, LAMP requires simple enzyme as the Bst DNA polymerase which we intended to improve using sythetic biology methods.</p><p>~<br>Comparison between different isothermal amplification methods[3]</p><p>However, the LAMP method has few limitations, such as LAMP amplifified DNA products can be visualized by gel electrophoresis or other detection methods. However, these methods may require the opening of the reaction tubes and increase the risk of carry-over contamination and complexity. A onepot reaction is helpful and the end-product can be visualized with</p><p>the naked eye, but a nakedeye method is not sensitive to variability among various samples and the detection limit is very high and the specificity can't be ensured. We prefered to use the lateral flow asssay to visualize the results as descreibed <span class="exturl" data-url="aHR0cDovL2JlbG93Lmlu">below.in</span> Lateral flow assay (LFA).</p><h3 id="choosing-marker-gene-and-design-of-primers-and-probes-2"><a class="anchor" href="#choosing-marker-gene-and-design-of-primers-and-probes-2">#</a> Choosing marker gene and design of primers and probes</h3><p>After bioinformatic analysis of the genome DNA using BLAST and referencing existing PCR detection method of <em>C.albicans</em> [6], we designed to target the ITS2 region of the 5.8S rDNA of <em>C.albicans</em>, and the primers and probe are as follow:</p><p>The LAMP primer sets were designed using Primer Explorer Ver. 4 (<span class="exturl" data-url="aHR0cDovL3ByaW1lcmV4cGxvcmVyLmpwL2Uv">http://primerexplorer.jp/e/</span>). For Lateral Flow Assay (LF) Assay detection, one detection probe was tagged with biotin at the 50 end, which has been entailed in th primer, and the other with FITC at 30 end. The newly designed probes were screened using the oligoanalyzer and BLAST tool (<span class="exturl" data-url="aHR0cDovL3d3dy5uY2JpLm5sbS5uaWgu">http://www.ncbi.nlm.nih.</span> gov/BLAST).</p><table><thead><tr><th style="text-align:left">Name</th><th style="text-align:left">Sequence</th><th style="text-align:left">type</th></tr></thead><tbody><tr><td style="text-align:left">F3</td><td style="text-align:left">TGGGTTTGCTTGAAAGACGG</td><td style="text-align:left">primer</td></tr><tr><td style="text-align:left">B3</td><td style="text-align:left">CTTCACTCGCCGCTACTG</td><td style="text-align:left"></td></tr><tr><td style="text-align:left">FIP</td><td style="text-align:left">CCGCCGCAAGCAATGTTTTTGGTAGTGGTAAGGCGGGATCG</td><td style="text-align:left"></td></tr><tr><td style="text-align:left">biotin-BIP</td><td style="text-align:left">ATCAGGTAGGACTACCCGCTGAAGGCAATCCCTGTTGGTTTC</td><td style="text-align:left"></td></tr><tr><td style="text-align:left">LF</td><td style="text-align:left">TTGACAATGGCTTAGGTCTA</td><td style="text-align:left"></td></tr><tr><td style="text-align:left">LB</td><td style="text-align:left">GCATATCAATAAGCGGAGGA</td><td style="text-align:left"></td></tr><tr><td style="text-align:left">FITC-probe</td><td style="text-align:left">ATTGCTTGCGGCGGTAACGTCC</td><td style="text-align:left">probe</td></tr></tbody></table><h2 id="lateral-flow-assaylfa-2"><a class="anchor" href="#lateral-flow-assaylfa-2">#</a> Lateral flow assay（LFA）</h2><p>The lateral flow immunoassay (LFA) is an immune chromatographic technique conducted on a lateral flow dipstick (LFD). LFA has been employed successfully to achieve point-of-care diagnostics [7]. LFA is specifific and sensitive, does not require a specifific instrument. Presently, rapid, sensitive, specifific, and affordable point-of-care tests are needed, especially in places that do not have access to laboratory services.<br>~<br>Schematic of the detection principle of LAMP-LFA</p><p>Amplicon specific self-designed tagged probes further confirmed species-specifific LAMP amplifification and ruled out misinterpretation due to any spurious amplifification. We demonstrated the simplicity and feasibility of this assay by detecting DNA extracted from Candida albicans and we also tested this assay under various backgrounds and processing conditions and high stability and sensitivity were obtained. From sampling to the detection of end-product was completed in less than 180 min and the only device that is needed in resource limited area is a thermos so that the detection can be point-of -care.</p><h1 id="applying-synthetic-biologydesigning-a-brand-new-t7-phage-expressing-system-to-lowering-the-cost-of-the-detection-2"><a class="anchor" href="#applying-synthetic-biologydesigning-a-brand-new-t7-phage-expressing-system-to-lowering-the-cost-of-the-detection-2">#</a> <strong>Applying synthetic biology：Designing a brand new T7 phage expressing system to lowering the cost of the detection</strong></h1><p></p><h2 id="a-new-idea-to-improve-the-enzyme-manufacturing-procedure-2"><a class="anchor" href="#a-new-idea-to-improve-the-enzyme-manufacturing-procedure-2">#</a> <strong>A new idea to improve the enzyme manufacturing procedure</strong></h2><p>In ordinary protein manufacturing procedure, we usually pay most attention to protein’s amount and activity, and one of the most common ways to achieve both high quantity and quality is linking tags to protein of interest and have it purified.</p><p></p><p>Purification of protein costs much, and it also leads to the problem of ‘how to transport and store these protein’. However, in our project, the target users are mainly those suffering poverty or living in underdeveloped regions. They may have no access to institution with lab environment, or just can’t afford the high cost. To cut these expense, we decide to use the extract of our engineered E.coli to directly catalyze the LAMP reaction. However, there’ll be lots of unrelated proteins in the extract disturbing the reaction, so we need our engineered E.coli to produce our protein of interest, the enzyme, as pure as possible. In another word, we need to raise the relative amount of our protein of interest.</p><p></p><p>After careful consideration, we decide to realize our idea by inhibiting the expression of all E.coli’s proteins except the protein of interest.</p><p></p><h2 id="ecoli-rnap-inhibitor-2"><a class="anchor" href="#ecoli-rnap-inhibitor-2">#</a> <strong>E.coli RNAP inhibitor</strong></h2><h3 id="background-2"><a class="anchor" href="#background-2">#</a> <strong>Background</strong></h3><p>During the transcription process of E.coli, transcription factors called ‘σ factors (sigma factors)’ are essential for guiding the RNA Polymerase (RNAP) to the promoters and initiating the transcription. We call RNAPs combined with σ factors ‘RNAP whole enzyme’. Each kinds of σ factors can specifically recognize corresponding promoters, thus realizing the overall regulation of gene expression at transcription level. There are at least seven kinds of σ factors that respectively be responsible to dominate the whole transcription process of host E.coli at different situations, including regular growth stages and extreme hazard. Among them, σ70 and σS are two major σ factors of E.coli. According to Abhishek Mazumder and Achillefs N. Kapanidis (2019) and Herb E. Schellhorn (2020), σ70 is the so-called ‘housekeeping’ σ factor which controls about 80% genes. In another word, when the E.coli is in ordinary environment, σ70 will always be the dominant σ factor. On the other hand, σS is the σ factor used by E.coli in order to counter starvation situation. In lab situation, we typically divide bacteria’s growth into four stages. According to the features of σ70 and σS, σS will take dominance at stationary stage while σ70 will show most significance at exponential stage. However, RNAP of phages, such as T7 phage’s RNAP, doesn’t necessarily rely on σ factors.</p><p></p><p>So it suddenly dawned on us that we can interfere σ factors to inhibit transcription and use phage’s transcription system to make our protein of interest immune to the inhibition. To interfere σ factors, we think of two ways. The first one is to let the engineered E.coli express dominant negative mutant of σ factor. But there’re so many kinds of σ factors, and their conserved regions are quite complicated. It’s hard to design an appropriate dominant negative mutant. Meanwhile, σ factors are quite huge as proteins, overexpress them is contradictory to our aim of reducing irrelevant proteins. The second way is to hinder the ability of σ factors. Then what we need to do is to find a proper inhibitor.</p><p></p><h3 id="inspired-by-mechanism-of-phages-2"><a class="anchor" href="#inspired-by-mechanism-of-phages-2">#</a> <strong>Inspired by mechanism of phages</strong></h3><p>Phage, well-known for being the virus infecting bacteria, is introduced in almost all biology text books in middle school. Briefly as the introduction is, they must tell you one phage’s key feature,’ Phages have the ability to shut bacteria down and turn them into factories that help phages replicate.’</p><p></p><p>In the field of synthesis biology, expressing systems of phages are good tools to help you achieve high protein yield. Among them, T7 system is one of the most mature expressing systems. However, the typical T7 expressing system merely utilizing T7 phage’s RNA polymerase, in another word, only materializing phage’s ability to turn bacteria into ‘factories’. Actually, there’s also huge potential in putting phage’s ability of ‘shutting down’ into expressing system.</p><p></p><p>One of T7’s shutting-down-methods is to inhibit the transcription initialization of bacteria by producing three proteins, respectively called gp2, gp5.7 and gp0.7. This process can redistribute the metabolism pressure of the host bacteria, letting them save mass and energy to produce what T7 phages want them to produce, meanwhile, reducing the relevant amount of other proteins for sure. That’s exactly the effects we want.</p><p></p><p>Among the three transcription inhibitors mentioned above, gp2 and gp5.7 are both binding proteins of RNAP whole-enzyme, while gp0.7 is a kinase with low specificity. In order to ensure the system’s robustness, we select gp2 and gp5.7 as candidates for constructing our gene circuit.</p><p></p><h3 id="gp2-2"><a class="anchor" href="#gp2-2">#</a> <strong>GP2</strong></h3><p>GP2 accounts for ‘T7 Gene Product 2’. It’s an gene product produced by E.coli infected by T7 phage at its early stage. According to Brian Bae et al.(2013), GP2 takes effects by simultaneously binding to RNAP β’ subunit as well as σ70 1.1domain, thus blocking the way DNA should be able to get in. By this process, all transcription process initiated by σ70 can be inhibited.</p><p>~</p><p>We are not the first team to propose GP2 in iGem community. In iGem 2018, Team: Imperial used T7 GP2 to hinder E.coli from growing without killing them, but they didn’t explain why GP2 can’t kill E.coli. We propose two hypothesis about this question. The first one is that, GP2 can inhibit the transcription of the mRNA of itself, thus reaching a balance. (这里最好想要一个 modeling 的图) The second is that, σS will take over σ70 after GP2 is expressed, and keep the cell surviving. Therefore, it’s necessary to introduce a σS-RNAP inhibitor to ensure the inhibition effect.</p><p></p><h3 id="gp57-2"><a class="anchor" href="#gp57-2">#</a> <strong>GP5.7</strong></h3><p>GP5.7 accounts for ‘T7 Gene Product 5.7’. It’s an gene product produced by E.coli infected by T7 phage at its middle stage. According to Aline Tabib-Salazar et al.(2018), GP5.7 takes effects by simultaneously binding to RNAP β’ subunit as well as σS R4 domain. By this process, all transcription process initiated by σS can be inhibited.</p><p>~</p><h3 id="σs-promoters-2"><a class="anchor" href="#σs-promoters-2">#</a> <strong>σS promoters</strong></h3><p>To prove the effect of GP2 and GP5.7. We need several σ70 promoters and σ70 promoters. While σ70 promoters are everywhere, σS promoters are super rare in iGem community. To solve this problem, we decide to introduce a set of σS promoters ourselves.</p><p>According to Regine Hengge-Aronis(2002), the regulation of σS doesn’t only includes the expression of σS. There’s a complicated regulation procedure. According to Anna Macia˛ g et al.(2011), σS promoters usually feature several UP elements that enable other factors to regulate them. In order to minimize the irrelevant factors, we only cut out about 35bp from those whole σS promoters. Since that might make the promoter lose its properties, we design an experiment to test and verify all the simplified σS promoters we propose.</p><p>σS promoters we propose include: CsiD, osmY, fic and dps, the sequences of which are shown below.</p><p>~<br>Gisela Becker and Regine Hengge-Aronis(2001) has analyzes several σS promoters sequence and found that they have quite conserved sequences in their simplest form. Since csiD and fic didn’t show as strong evidence of being σS promoters as dps and osmY, we decide to improve their sequence manually by introducing spot mutation following Gisela Becker and Regine Hengge-Aronis’s results. The sequence of improved promoters are shown below.</p><p>~</p><h1 id="designing-fusion-proteins-to-enhance-bst-dna-polymerase-activity-2"><a class="anchor" href="#designing-fusion-proteins-to-enhance-bst-dna-polymerase-activity-2">#</a> <strong>Designing fusion proteins to enhance Bst DNA polymerase activity</strong></h1><h2 id="about-the-double-stranded-binding-protein-sso7d-2"><a class="anchor" href="#about-the-double-stranded-binding-protein-sso7d-2">#</a> <strong>About the double-stranded binding protein Sso7d</strong></h2><p>Sso7d is a kind of double-stranded binding protein, and it be used widely to improve the activity of polymerase[[](file:///C:/Users/25411/Desktop/%E6%96%B0%E5%BB%BA%20DOCX%20%E6%96%87%E6%A1%A3%20(3).docx#_edn1)[7](file:///C:/Users/25411/Desktop/%E6%96%B0%E5%BB%BA%20DOCX%20%E6%96%87%E6%A1%A3%20(3).docx#_edn1)[]](file:///C:/Users/25411/Desktop/%E6%96%B0%E5%BB%BA%20DOCX%20%E6%96%87%E6%A1%A3%20(3).docx#_edn1). The DNA structure changes at different concentrations of Sso7d and depends on the reaction time. At relatively low concentrations of Sso7d, DNA strands form a kinked structure. When the concentration of Sso7d increases, DNA loops appear. Finally, DNA becomes a dense nuclear structure at high concentrations of Sso7d. If the time of interaction between Sso7d and DNA increases, the DNA structure tends to be more compact. High concentrations of Sso7d are important for the compact structure of DNA[[](file:///C:/Users/25411/Desktop/%E6%96%B0%E5%BB%BA%20DOCX%20%E6%96%87%E6%A1%A3%20(3).docx#_edn2)[8](file:///C:/Users/25411/Desktop/%E6%96%B0%E5%BB%BA%20DOCX%20%E6%96%87%E6%A1%A3%20(3).docx#_edn2)[]](file:///C:/Users/25411/Desktop/%E6%96%B0%E5%BB%BA%20DOCX%20%E6%96%87%E6%A1%A3%20(3).docx#_edn2). Therefore, Sso7d has an affinity for double-stranded binding proteins and thus recognizes the annealed double-strand, allowing the polymerase to quickly find the extension site, thus effectively increasing the extension speed of the DNA polymerase.</p><p></p><h2 id="a-new-idea-to-enhance-the-activity-of-bst-dna-polymerase-2"><a class="anchor" href="#a-new-idea-to-enhance-the-activity-of-bst-dna-polymerase-2">#</a> <strong>A new idea to enhance the activity of Bst DNA polymerase</strong></h2><p>It has been widely reported in the past that the activity associated with Pfu DNA polymerase can be greatly improved by adding the double-stranded binding protein Sso7d1 to the C-terminus of Pfu DNA polymerase. Meanwhile, our laboratory has done experiments to improve the activity of DNA polymerase Ⅱ by adding double-stranded binding protein in the past and has accumulated more experience[[](file:///C:/Users/25411/Desktop/%E6%96%B0%E5%BB%BA%20DOCX%20%E6%96%87%E6%A1%A3%20(3).docx#_edn3)[9](file:///C:/Users/25411/Desktop/%E6%96%B0%E5%BB%BA%20DOCX%20%E6%96%87%E6%A1%A3%20(3).docx#_edn3)[]](file:///C:/Users/25411/Desktop/%E6%96%B0%E5%BB%BA%20DOCX%20%E6%96%87%E6%A1%A3%20(3).docx#_edn3). In previous studies, Sso7d was suggested to synthesize fusion proteins with DNA polymerase A or DNA polymerase B to largely increase polymerase activity, but no studies have been conducted on the changes in the activity of DNA polymerase Ⅰ and Sso7d fusion proteins. Even so, we still speculated that the activity of Bst DNA polymerase might be improved by linking double-stranded binding proteins, and selected four double-stranded binding proteins from the same hyperthermophilic archaeon Sulfolobus solfataricus, Sso7d, Sso10b, DbpA and albA1, for our experiments. Based on the results of previous modifications of pfu DNA polymerase, we predicted that the bst-sso7d fusion protein might possess the highest activity.</p><p></p><h2 id="our-selection-of-binding-proteins-2"><a class="anchor" href="#our-selection-of-binding-proteins-2">#</a> <strong>Our selection of binding proteins</strong></h2><p>We selected six binding proteins for our experiments ( two for single-stranded binding protein and four for double-stranded binding protein). We asseumed that just as double-stranded binding proteins can bind annealed double-stranded DNA, single-stranded binding proteins can also bind denatured single-stranded DNA and thus improve the amplification efficiency of DNA polymerase. In addition to this, we designed two possible twelve-component experiments by ligating the binding proteins to the N-terminal and C-terminal ends of Bst DNA polymerase, respectively, for the experiments. In designing the fusion protein, we chose the flexible linker ((G2S)3) as the linker of our fusion protein.</p><h3 id="according-to-the-consequence-of-our-experiments-our-modification-actually-works-as-we-expected-2"><a class="anchor" href="#according-to-the-consequence-of-our-experiments-our-modification-actually-works-as-we-expected-2">#</a> <strong>According to the consequence of our experiments, our modification actually works as we expected.</strong></h3><h2 id="references-2"><a class="anchor" href="#references-2">#</a> References</h2><p>[1] Poulain D. (2015). Candida albicans, plasticity and pathogenesis. <em>Critical reviews in microbiology</em>, <em>41</em>(2), 208–217. <span class="exturl" data-url="aHR0cHM6Ly9kb2kub3JnLzEwLjMxMDkvMTA0MDg0MVguMjAxMy44MTM5MDQ=">https://doi.org/10.3109/1040841X.2013.813904</span></p><p>[2] Elie, C. M., Lott, T. J., Reiss, E., &amp; Morrison, C. J. (1998). Rapid identification of Candida species with species-specific DNA probes. <em>Journal of clinical microbiology</em>, <em>36</em>(11), 3260–3265. <span class="exturl" data-url="aHR0cHM6Ly9kb2kub3JnLzEwLjExMjgvSkNNLjM2LjExLjMyNjAtMzI2NS4xOTk4">https://doi.org/10.1128/JCM.36.11.3260-3265.1998</span></p><p>[3] Zhao, Y., Chen, F., Li, Q., Wang, L., &amp; Fan, C. (2015). Isothermal Amplification of Nucleic Acids. <em>Chemical reviews</em>, 115(22), 12491–12545. <span class="exturl" data-url="aHR0cHM6Ly9kb2kub3JnLzEwLjEwMjEvYWNzLmNoZW1yZXYuNWIwMDQyOA==">https://doi.org/10.1021/acs.chemrev.5b00428</span></p><p>[4] Wong, Y. P., Othman, S., Lau, Y. L., Radu, S., &amp; Chee, H. Y. (2018). Loop-mediated isothermal amplification (LAMP): a versatile technique for detection of micro-organisms. <em>Journal of applied microbiology</em>, <em>124</em>(3), 626–643. <span class="exturl" data-url="aHR0cHM6Ly9kb2kub3JnLzEwLjExMTEvamFtLjEzNjQ3">https://doi.org/10.1111/jam.13647</span></p><p>[5] Fallahi, S., Babaei, M., Rostami, A. <em>et al.</em> Diagnosis of <em>Candida albicans</em>: conventional diagnostic methods compared to the loop-mediated isothermal amplification (LAMP) assay. <em>Arch Microbiol</em> **202,**275–282 (2020). <span class="exturl" data-url="aHR0cHM6Ly9kb2kub3JnLzEwLjEwMDcvczAwMjAzLTAxOS0wMTczNi03">https://doi.org/10.1007/s00203-019-01736-7</span></p><p>[6] Koczula, K. M., &amp; Gallotta, A. (2016). Lateral flow assays. <em>Essays in biochemistry</em>, <em>60</em>(1), 111–120. <span class="exturl" data-url="aHR0cHM6Ly9kb2kub3JnLzEwLjEwNDIvRUJDMjAxNTAwMTI=">https://doi.org/10.1042/EBC20150012</span></p><p>[[7]](file:///C:/Users/25411/Desktop/%E6%96%B0%E5%BB%BA%20DOCX%20%E6%96%87%E6%A1%A3%20(3).docx#_ednref1) Wang Y, Prosen DE, Mei L, Sullivan JC, Finney M, Vander Horn PB. A novel strategy to engineer DNA polymerases for enhanced processivity and improved performance in vitro. Nucleic Acids Res. 2004 Feb 18;32(3):1197-207. doi: 10.1093/nar/gkh271. PMID: 14973201; PMCID: PMC373405.</p><p>[[8]](file:///C:/Users/25411/Desktop/%E6%96%B0%E5%BB%BA%20DOCX%20%E6%96%87%E6%A1%A3%20(3).docx#_ednref2) Kalichuk, V., Béhar, G., Renodon-Cornière, A. et al. The archaeal “7 kDa DNA-binding” proteins: extended characterization of an old gifted family. Sci Rep 6, 37274 (2016). <span class="exturl" data-url="aHR0cHM6Ly9kb2kub3JnLzEwLjEwMzgvc3JlcDM3Mjc0">https://doi.org/10.1038/srep37274</span></p><p>[[9]](file:///C:/Users/25411/Desktop/%E6%96%B0%E5%BB%BA%20DOCX%20%E6%96%87%E6%A1%A3%20(3).docx#_ednref3) Cao S-C, Qiu L-Z. Study of DNA binding protein DbpA affecting the performance of DNA polymerase[J]. Journal of Fudan:Natural Science Edition, 2015, 54(4):469-477.</p></div><footer><div class="meta"><span class="item"><span class="icon"><i class="ic i-calendar-check"></i> </span><span class="text">Edited on</span> <time title="Modified: 2021-10-22 11:54:30" itemprop="dateModified" datetime="2021-10-22T11:54:30+08:00">2021-10-22</time></span></div></footer></article></div><div class="wrap" id="comments"></div></div><div id="sidebar"><div class="inner"><div class="panels"><div class="inner"><div class="contents panel pjax" data-title="Contents"><ol class="toc"><li class="toc-item toc-level-1"><a class="toc-link" href="#inspiration-and-original-design"><span class="toc-number">1.</span> <span class="toc-text">Inspiration and Original Design</span></a></li><li class="toc-item toc-level-1"><a class="toc-link" href="#detection-of-candida-albicans"><span class="toc-number">2.</span> <span class="toc-text">Detection of Candida Albicans</span></a></li><li class="toc-item toc-level-1"><a class="toc-link" href="#current-detection-methods-and-their-weaknesses"><span class="toc-number">3.</span> <span class="toc-text">Current detection methods and their weaknesses</span></a></li><li class="toc-item toc-level-1"><a class="toc-link" href="#the-goal-of-our-project"><span class="toc-number">4.</span> <span class="toc-text">The goal of our project</span></a></li><li class="toc-item toc-level-1"><a class="toc-link" href="#overview"><span class="toc-number">5.</span> <span class="toc-text">Overview</span></a></li><li class="toc-item toc-level-1"><a class="toc-link" href="#isothermal-amplification"><span class="toc-number">6.</span> <span class="toc-text">Isothermal amplification</span></a><ol class="toc-child"><li class="toc-item toc-level-2"><a class="toc-link" href="#loop-mediated-isothermal-amplification-lamp"><span class="toc-number">6.1.</span> <span class="toc-text">Loop-mediated isothermal amplification (LAMP)</span></a><ol class="toc-child"><li class="toc-item toc-level-3"><a class="toc-link" href="#why-did-we-choose-lamp"><span class="toc-number">6.1.1.</span> <span class="toc-text">Why did we choose LAMP?</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#choosing-marker-gene-and-design-of-primers-and-probes"><span class="toc-number">6.1.2.</span> <span class="toc-text">Choosing marker gene and design of primers and probes</span></a></li></ol></li><li class="toc-item toc-level-2"><a class="toc-link" href="#lateral-flow-assaylfa"><span class="toc-number">6.2.</span> <span class="toc-text">Lateral flow assay（LFA）</span></a></li></ol></li><li class="toc-item toc-level-1"><a class="toc-link" href="#applying-synthetic-biologydesigning-a-brand-new-t7-phage-expressing-system-to-lowering-the-cost-of-the-detection"><span class="toc-number">7.</span> <span class="toc-text">Applying synthetic biology：Designing a brand new T7 phage expressing system to lowering the cost of the detection</span></a><ol class="toc-child"><li class="toc-item toc-level-2"><a class="toc-link" href="#a-new-idea-to-improve-the-enzyme-manufacturing-procedure"><span class="toc-number">7.1.</span> <span class="toc-text">A new idea to improve the enzyme manufacturing procedure</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#ecoli-rnap-inhibitor"><span class="toc-number">7.2.</span> <span class="toc-text">E.coli RNAP inhibitor</span></a><ol class="toc-child"><li class="toc-item toc-level-3"><a class="toc-link" href="#background"><span class="toc-number">7.2.1.</span> <span class="toc-text">Background</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#inspired-by-mechanism-of-phages"><span class="toc-number">7.2.2.</span> <span class="toc-text">Inspired by mechanism of phages</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#gp2"><span class="toc-number">7.2.3.</span> <span class="toc-text">GP2</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#gp57"><span class="toc-number">7.2.4.</span> <span class="toc-text">GP5.7</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#%CF%83s-promoters"><span class="toc-number">7.2.5.</span> <span class="toc-text">σS promoters</span></a></li></ol></li></ol></li><li class="toc-item toc-level-1"><a class="toc-link" href="#designing-fusion-proteins-to-enhance-bst-dna-polymerase-activity"><span class="toc-number">8.</span> <span class="toc-text">Designing fusion proteins to enhance Bst DNA polymerase activity</span></a><ol class="toc-child"><li class="toc-item toc-level-2"><a class="toc-link" href="#about-the-double-stranded-binding-protein-sso7d"><span class="toc-number">8.1.</span> <span class="toc-text">About the double-stranded binding protein Sso7d</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#a-new-idea-to-enhance-the-activity-of-bst-dna-polymerase"><span class="toc-number">8.2.</span> <span class="toc-text">A new idea to enhance the activity of Bst DNA polymerase</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#our-selection-of-binding-proteins"><span class="toc-number">8.3.</span> <span class="toc-text">Our selection of binding proteins</span></a><ol class="toc-child"><li class="toc-item toc-level-3"><a class="toc-link" href="#according-to-the-consequence-of-our-experiments-our-modification-actually-works-as-we-expected"><span class="toc-number">8.3.1.</span> <span class="toc-text">According to the consequence of our experiments, our modification actually works as we expected.</span></a></li></ol></li><li class="toc-item toc-level-2"><a class="toc-link" href="#references"><span class="toc-number">8.4.</span> <span class="toc-text">References</span></a></li></ol></li><li class="toc-item toc-level-1"><a class="toc-link" href="#overview-2"><span class="toc-number">9.</span> <span class="toc-text">Overview</span></a></li><li class="toc-item toc-level-1"><a class="toc-link" href="#isothermal-amplification-2"><span class="toc-number">10.</span> <span class="toc-text">Isothermal amplification</span></a><ol class="toc-child"><li class="toc-item toc-level-2"><a class="toc-link" href="#loop-mediated-isothermal-amplification-lamp-2"><span class="toc-number">10.1.</span> <span class="toc-text">Loop-mediated isothermal amplification (LAMP)</span></a><ol class="toc-child"><li class="toc-item toc-level-3"><a class="toc-link" href="#why-did-we-choose-lamp-2"><span class="toc-number">10.1.1.</span> <span class="toc-text">Why did we choose LAMP?</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#choosing-marker-gene-and-design-of-primers-and-probes-2"><span class="toc-number">10.1.2.</span> <span class="toc-text">Choosing marker gene and design of primers and probes</span></a></li></ol></li><li class="toc-item toc-level-2"><a class="toc-link" href="#lateral-flow-assaylfa-2"><span class="toc-number">10.2.</span> <span class="toc-text">Lateral flow assay（LFA）</span></a></li></ol></li><li class="toc-item toc-level-1"><a class="toc-link" href="#applying-synthetic-biologydesigning-a-brand-new-t7-phage-expressing-system-to-lowering-the-cost-of-the-detection-2"><span class="toc-number">11.</span> <span class="toc-text">Applying synthetic biology：Designing a brand new T7 phage expressing system to lowering the cost of the detection</span></a><ol class="toc-child"><li class="toc-item toc-level-2"><a class="toc-link" href="#a-new-idea-to-improve-the-enzyme-manufacturing-procedure-2"><span class="toc-number">11.1.</span> <span class="toc-text">A new idea to improve the enzyme manufacturing procedure</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#ecoli-rnap-inhibitor-2"><span class="toc-number">11.2.</span> <span class="toc-text">E.coli RNAP inhibitor</span></a><ol class="toc-child"><li class="toc-item toc-level-3"><a class="toc-link" href="#background-2"><span class="toc-number">11.2.1.</span> <span class="toc-text">Background</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#inspired-by-mechanism-of-phages-2"><span class="toc-number">11.2.2.</span> <span class="toc-text">Inspired by mechanism of phages</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#gp2-2"><span class="toc-number">11.2.3.</span> <span class="toc-text">GP2</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#gp57-2"><span class="toc-number">11.2.4.</span> <span class="toc-text">GP5.7</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#%CF%83s-promoters-2"><span class="toc-number">11.2.5.</span> <span class="toc-text">σS promoters</span></a></li></ol></li></ol></li><li class="toc-item toc-level-1"><a class="toc-link" href="#designing-fusion-proteins-to-enhance-bst-dna-polymerase-activity-2"><span class="toc-number">12.</span> <span class="toc-text">Designing fusion proteins to enhance Bst DNA polymerase activity</span></a><ol class="toc-child"><li class="toc-item toc-level-2"><a class="toc-link" href="#about-the-double-stranded-binding-protein-sso7d-2"><span class="toc-number">12.1.</span> <span class="toc-text">About the double-stranded binding protein Sso7d</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#a-new-idea-to-enhance-the-activity-of-bst-dna-polymerase-2"><span class="toc-number">12.2.</span> <span class="toc-text">A new idea to enhance the activity of Bst DNA polymerase</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#our-selection-of-binding-proteins-2"><span class="toc-number">12.3.</span> <span class="toc-text">Our selection of binding proteins</span></a><ol class="toc-child"><li class="toc-item toc-level-3"><a class="toc-link" href="#according-to-the-consequence-of-our-experiments-our-modification-actually-works-as-we-expected-2"><span class="toc-number">12.3.1.</span> <span class="toc-text">According to the consequence of our experiments, our modification actually works as we expected.</span></a></li></ol></li><li class="toc-item toc-level-2"><a class="toc-link" href="#references-2"><span class="toc-number">12.4.</span> <span class="toc-text">References</span></a></li></ol></li></ol></div><div class="related panel pjax" data-title="Related"></div><div class="overview panel" data-title="Overview"><div class="author" itemprop="author" itemscope itemtype="http://schema.org/Person"><img class="image" itemprop="image" alt="Tom Ghost Smith" data-src="/images/avatar.jpg"><p class="name" itemprop="name">Tom Ghost Smith</p><div class="description" itemprop="description">描述 123</div></div><nav class="state"><div class="item posts"><a href="/archives/"><span class="count">26</span> <span class="name">posts</span></a></div><div class="item categories"><a href="/categories/"><span class="count">8</span> <span class="name">categories</span></a></div><div class="item tags"><a href="/tags/"><span class="count">5</span> <span class="name">tags</span></a></div></nav><div class="social"><span class="exturl item github" data-url="aHR0cHM6Ly9naXRodWIuY29tL1RvbUdob3N0U21pdGg=" title="https:&#x2F;&#x2F;github.com&#x2F;TomGhostSmith"><i class="ic i-github"></i></span> <span class="exturl item twitter" data-url="aHR0cHM6Ly90d2l0dGVyLmNvbS95b3VybmFtZQ==" title="https:&#x2F;&#x2F;twitter.com&#x2F;yourname"><i class="ic i-twitter"></i></span> <span class="exturl item zhihu" data-url="aHR0cHM6Ly93d3cuemhpaHUuY29tL3Blb3BsZS95b3VybmFtZQ==" title="https:&#x2F;&#x2F;www.zhihu.com&#x2F;people&#x2F;yourname"><i class="ic i-zhihu"></i></span> <span class="exturl item music" data-url="aHR0cHM6Ly9tdXNpYy4xNjMuY29tLyMvdXNlci9ob21lP2lkPXlvdXJpZA==" title="https:&#x2F;&#x2F;music.163.com&#x2F;#&#x2F;user&#x2F;home?id&#x3D;yourid"><i class="ic i-cloud-music"></i></span> <span class="exturl item weibo" data-url="aHR0cHM6Ly93ZWliby5jb20veW91cm5hbWU=" title="https:&#x2F;&#x2F;weibo.com&#x2F;yourname"><i class="ic i-weibo"></i></span> <span class="exturl item about" data-url="aHR0cHM6Ly9hYm91dC5tZS95b3VybmFtZQ==" title="https:&#x2F;&#x2F;about.me&#x2F;yourname"><i class="ic i-address-card"></i></span></div><ul class="menu"><li class="item"><a href="/" rel="section"><i class="ic i-home"></i>Home</a></li><li class="item dropdown"><a href="javascript:void(0);"><i class="ic i-feather"></i>Posts</a><ul class="submenu"><li class="item"><a href="/archives/" rel="section"><i class="ic i-list-alt"></i>Archives</a></li><li class="item"><a href="/categories/" rel="section"><i class="ic i-th"></i>Categories</a></li><li class="item"><a href="/tags/" rel="section"><i class="ic i-tags"></i>Tags</a></li></ul></li><li class="item"><a href="/friends/" rel="section"><i class="ic i-heart"></i>Friends</a></li></ul></div></div></div><ul id="quick"><li class="prev pjax"></li><li class="up"><i class="ic i-arrow-up"></i></li><li class="down"><i class="ic i-arrow-down"></i></li><li class="next pjax"></li><li class="percent"></li></ul></div></div><div class="dimmer"></div></div></main><footer id="footer"><div class="inner"><div class="widgets"><div class="rpost pjax"><h2>Random Posts</h2><ul><li class="item"><div class="breadcrumb"></div><span><a href="/Page2/" title="Page2">Page2</a></span></li><li class="item"><div class="breadcrumb"></div><span><span class="exturl" data-url="aHR0cDovL3d3dy5nb29nbGUuY29tLw==" title="Link Post">Link Post<i class="ic i-link-alt"></i></span></span></li><li class="item"><div class="breadcrumb"></div><span><a href="/gallery-post/" title="Gallery Post">Gallery Post</a></span></li><li class="item"><div class="breadcrumb"><a href="/categories/computer-science/" title="In 计算机科学">计算机科学</a> <i class="ic i-angle-right"></i> <a href="/categories/computer-science/java/" title="In Java">Java</a> <i class="ic i-angle-right"></i> <a href="/categories/computer-science/java/course-1/" title="In 零基础学 Java 语言 - 浙江大学 - 翁恺">零基础学 Java 语言 - 浙江大学 - 翁恺</a></div><span><a href="/computer-science/java/course-1/week-2/" title="第2周 判断">第2周 判断</a></span></li><li class="item"><div class="breadcrumb"></div><span><a href="/long-title/" title="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam justo turpis, tincidunt ac convallis id.">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam justo turpis, tincidunt ac convallis id.</a></span></li><li class="item"><div class="breadcrumb"></div><span><a href="/excerpts/" title="Excerpts">Excerpts</a></span></li><li class="item"><div class="breadcrumb"><a href="/categories/Foo/" title="In Foo">Foo</a> <i class="ic i-angle-right"></i> <a href="/categories/Foo/Bar/" title="In Bar">Bar</a> <i class="ic i-angle-right"></i> <a href="/categories/Foo/Bar/Baz/" title="In Baz">Baz</a></div><span><a href="/categories/" title="Categories">Categories</a></span></li><li class="item"><div class="breadcrumb"></div><span><a href="/%E4%B8%AD%E6%96%87%E6%B8%AC%E8%A9%A6/" title="中文測試">中文測試</a></span></li><li class="item"><div class="breadcrumb"><a href="/categories/computer-science/" title="In 计算机科学">计算机科学</a> <i class="ic i-angle-right"></i> <a href="/categories/computer-science/note/" title="In 二进制杂谈">二进制杂谈</a> <i class="ic i-angle-right"></i> <a href="/categories/computer-science/note/Theme-Shoka-Documentation/" title="In Theme Shoka Documentation">Theme Shoka Documentation</a></div><span><a href="/computer-science/note/theme-shoka-doc/display/" title="Step.3 界面显示">Step.3 界面显示</a></span></li><li class="item"><div class="breadcrumb"><a href="/categories/computer-science/" title="In 计算机科学">计算机科学</a> <i class="ic i-angle-right"></i> <a href="/categories/computer-science/note/" title="In 二进制杂谈">二进制杂谈</a> <i class="ic i-angle-right"></i> <a href="/categories/computer-science/note/Theme-Shoka-Documentation/" title="In Theme Shoka Documentation">Theme Shoka Documentation</a></div><span><a href="/computer-science/note/theme-shoka-doc/dependents/" title="Step.1 依赖插件">Step.1 依赖插件</a></span></li></ul></div><div><h2>Recent Comments</h2><ul class="leancloud-recent-comment"></ul></div></div><div class="status"><div class="copyright">&copy; 2010 – <span itemprop="copyrightYear">2021</span> <span class="with-love"><i class="ic i-sakura rotate"></i> </span><span class="author" itemprop="copyrightHolder">Tom Ghost Smith @ Yume Shoka</span></div><div class="count"><span class="post-meta-item-icon"><i class="ic i-chart-area"></i> </span><span title="Symbols count total">NaNm words</span> <span class="post-meta-divider">|</span> <span class="post-meta-item-icon"><i class="ic i-coffee"></i> </span><span title="Reading time total">NaN:aN</span></div><div class="powered-by">Powered by <span class="exturl" data-url="aHR0cHM6Ly9oZXhvLmlv">Hexo</span> & Theme.<span class="exturl" data-url="aHR0cHM6Ly9naXRodWIuY29tL2FtZWhpbWUvaGV4by10aGVtZS1zaG9rYQ==">Shoka</span></div></div></div></footer></div><script data-config type="text/javascript">var LOCAL={path:"wiki/Description.html",favicon:{show:"（●´3｀●）Goooood",hide:"(´Д｀)Booooom"},search:{placeholder:"Search for Posts",empty:"We didn't find any results for the search: ${query}",stats:"${hits} results found in ${time} ms"},valine:!0,fancybox:!0,copyright:'Copied to clipboard successfully! <br> All articles in this blog are licensed under <i class="ic i-creative-commons"></i>BY-NC-SA.',ignores:[function(e){return e.includes("#")},function(e){return new RegExp(LOCAL.path+"$").test(e)}]}</script><script src="https://cdn.polyfill.io/v2/polyfill.js"></script><script src="//cdn.jsdelivr.net/combine/npm/pace-js@1.0.2/pace.min.js,npm/pjax@0.2.8/pjax.min.js,npm/whatwg-fetch@3.4.0/dist/fetch.umd.min.js,npm/animejs@3.2.0/lib/anime.min.js,npm/algoliasearch@4/dist/algoliasearch-lite.umd.js,npm/instantsearch.js@4/dist/instantsearch.production.min.js,npm/lozad@1/dist/lozad.min.js,npm/quicklink@2/dist/quicklink.umd.js"></script><script src="/js/app.js?v=0.2.5"></script></body></html>