<!DOCTYPE html><html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=2"><meta name="theme-color" content="#FFF"><link rel="alternate" type="application/rss+xml" title="Title" href="https://2021.igem.org/Template:Fudan/Others/rss_xml"><link rel="alternate" type="application/atom+xml" title="Title" href="https://2021.igem.org/Template:Fudan/Others/atom_xml"><link rel="alternate" type="application/json" title="Title" href="https://2021.igem.org/Template:Fudan/Others/feed_json"><link rel="stylesheet" type="text/css" href="https://2021.igem.org/Template:Fudan/CSS/main_css?action=raw&ctype=text/css"><meta name="generator" content="Hexo 5.4.0"></head><body><div id="loading"><img src="https://2021.igem.org/wiki/images/b/b9/T--Fudan--Loading.gif" alt=""></div><div id="container"><header id="header"><div class="inner"><div id="brand"><div class="pjax"><h1 itemprop="name headline">Modeling</h1></div></div><nav id="nav"><div class="inner"><div class="toggle"><div class="lines" aria-label="Toggle navigation bar"><span class="line"></span> <span class="line"></span> <span class="line"></span></div></div><ul class="menu"><li class="item title"><a target="_blank" href="https://2021.igem.org/Team:Fudan" rel="start noopener">Home</a></li><li class="item dropdown"><a href="javascript:void(0);">Team</a><ul class="submenu"><li class="item"><a href="/archives/" rel="section">Team</a></li><li class="item"><a href="/categories/" rel="section">Attributions</a></li><li class="item"><a href="/tags/" rel="section">Collaborations</a></li></ul></li><li class="item dropdown"><a href="javascript:void(0);">Project</a><ul class="submenu"><li class="item"><a href="/archives/" rel="section">Communication</a></li><li class="item"><a href="/categories/" rel="section">Contribution</a></li><li class="item"><a href="/tags/" rel="section">Description</a></li><li class="item"><a href="/tags/" rel="section">Engineering</a></li><li class="item"><a href="/tags/" rel="section">Experiments</a></li><li class="item"><a href="/tags/" rel="section">Implementation</a></li><li class="item"><a href="/tags/" rel="section">Notebook</a></li><li class="item"><a href="/tags/" rel="section">Partnership</a></li><li class="item"><a href="/tags/" rel="section">Proof Of Concept</a></li><li class="item"><a href="/tags/" rel="section">Results</a></li></ul></li><li class="item dropdown"><a href="javascript:void(0);">Parts</a><ul class="submenu"><li class="item"><a href="/archives/" rel="section">Part Collection</a></li><li class="item"><a href="/categories/" rel="section">Parts</a></li><li class="item"><a href="/tags/" rel="section">Improve</a></li></ul></li><li class="item"><a href="javascript:void(0);">Safety</a></li><li class="item dropdown"><a href="javascript:void(0);">Human Practice</a><ul class="submenu"><li class="item"><a href="/archives/" rel="section">1</a></li><li class="item"><a href="/categories/" rel="section">2</a></li><li class="item"><a href="/tags/" rel="section">3</a></li></ul></li><li class="item dropdown"><a href="javascript:void(0);">Awards</a><ul class="submenu"><li class="item"><a href="/archives/" rel="section">Education</a></li><li class="item"><a href="/categories/" rel="section">Entrepreneurship</a></li><li class="item"><a href="/tags/" rel="section">Hardware</a></li><li class="item"><a href="/tags/" rel="section">Inclusivity</a></li><li class="item"><a href="/tags/" rel="section">Measurement</a></li><li class="item"><a href="/tags/" rel="section">Model</a></li><li class="item"><a href="/tags/" rel="section">Plant</a></li><li class="item"><a href="/tags/" rel="section">Software</a></li><li class="item"><a href="/tags/" rel="section">Sustainable</a></li></ul></li><li class="item"><a href="javascript:void(0);">Poster</a></li></ul><ul class="right"><li class="item theme"><i class="ic i-sun"></i></li></ul></div></nav></div><div id="imgs" class="pjax"><ul><li class="item"><img src="https://tva2.sinaimg.cn/large/6833939bly1giclgrvbd6j20zk0m8qv5.jpg"></li><li class="item"><img src="https://tva2.sinaimg.cn/large/6833939bly1giclfw2t96j20zk0m8x6p.jpg"></li><li class="item"><img src="https://tva2.sinaimg.cn/large/6833939bly1gipewr8iypj20zk0m8b29.jpg"></li><li class="item"><img src="https://tva2.sinaimg.cn/large/6833939bly1gicitf0kl1j20zk0m87fe.jpg"></li><li class="item"><img src="https://tva2.sinaimg.cn/large/6833939bly1giclhnx9glj20zk0m8npd.jpg"></li><li class="item"><img src="https://tva2.sinaimg.cn/large/6833939bly1gipesng5oej20zk0m87d4.jpg"></li></ul></div></header><div id="waves"><svg class="waves" viewBox="0 24 150 28" preserveAspectRatio="none" shape-rendering="auto"><defs><path id="gentle-wave" d="M-160 44c30 0 58-18 88-18s 58 18 88 18 58-18 88-18 58 18 88 18 v44h-352z"/></defs><g class="parallax"><use xlink:href="#gentle-wave" x="48" y="0"/><use xlink:href="#gentle-wave" x="48" y="3"/><use xlink:href="#gentle-wave" x="48" y="5"/><use xlink:href="#gentle-wave" x="48" y="7"/></g></svg></div><main><div class="inner"><div id="main" class="pjax"><div class="page wrap"><article itemscope itemtype="" class="post block" lang="en"><div class="body md" itemprop="articleBody"><h1 id="key-problems"><a class="anchor" href="#key-problems">#</a> Key Problems</h1><p>Our T7 expression model enables host bacterial enzyme production with 3 basic parts: genes encoding T7 phage polymerase, inhibitor protein gp, and target Bst (GFP in our lab experiments) with T7P promoter.</p><p>Gp2 and gp5.7 are two inhibitors of host bacteria genes by binding specifically to transcription factors σ70 and σs. Thus, to purify our target protein, it’s necessary to downregulate the endogenous protein of E. coli. We designed two ways to express Igp5.7 which will inhibit ss, and the first aim of our model is to demonstrate the better one.</p><p>At the same time, T7P which can produce our target protein Bst is influenced by the concentration of s70 and ss. If the concentration of the two transcription factors decreases, then we will expect that the concentration of T7P mRNA would decrease along with them and then the T7P concentration. Here we can expect that if the duration of induction is long enough, and then the decrease of endogenous transcription will affect our target protein which can only be expressed by T7P. the second aim of our modeling is to find a proper time to harvest Bst post-induction.</p><p><strong>In Brief, Two Main Aims for Our Model:</strong></p><ol><li>High Expression and Purity of Target Protein Bst (written GFP in our model).</li><li>Better Way to Downregulate Endogenous Transcription</li></ol><h1 id="key-parameters-and-processes"><a class="anchor" href="#key-parameters-and-processes">#</a> Key Parameters and Processes</h1><p>To accurately describe our reactions and system behavior, our model is based on Kinetic Equations and Modeling. Our model basically involves four parts: 1. Kinetic Constant, 2. Key Variable, 3. Key Process, 4. Key Differential Equation.</p><h2 id="key-constants-in-model"><a class="anchor" href="#key-constants-in-model">#</a> Key Constants in Model</h2><p><img data-src="https://2021.igem.org/wiki/images/3/37/T--Fudan--modeling6.jpg" alt="pic"><br><img data-src="https://2021.igem.org/wiki/images/7/71/T--Fudan--modeling13.jpg" alt="pic"><br><strong>Table 2.1. List of all parameters used and our rationale. Kinetic constants for matching processes are divided into 7 categories: transcription, translation, dissociation, binding(inhibition), degradation, active transportation, and others. Assumptions and references are listed below in 1.1 and 1.2.</strong></p><h3 id="assumptions"><a class="anchor" href="#assumptions">#</a> Assumptions</h3><ol><li>The materials given are sufficient to conduct all reactions mentioned (peptide, nucleotide, ribosome, etc.)</li><li>Quasi-steady state. We assumed that the material used for production is sufficient for our bacteria. Thus, to steady the whole model, long-term induction or expression of proteins must converge to fixed values.</li><li>Translation kinetic constants for all proteins vary in a negligible range and are assumed to be equal in our model. E.g. Kσs = Kσ70 = KP1 [4]</li><li>Transcription kinetic constants for all mRNA with the same promoter vary in negligible range. E.g. K70MP1 = K70Igp2 etc.</li><li>Degradation kinetic constants for all protein/mRNA vary in negligible range. E.g., λ mRNA1 = λ mRNA2, λ Protein1 = λ Protein 2 etc.</li><li>Elements involved in reaction don’t work in quantization. In our model, we assumed a value, however small(e.g. 1.03 * 10^-5 or even smaller), for every variable. This is how during simulation we can find the curve smooth and derivable at every point. Numeric floating points in simulation can also participate in biochemical reactions.</li><li>Bacteria stress in metabolism is an abstract factor often overlooked in model simulation We visualize the stress as the rate of endogenous protein expression. Decrease of the protein to some extend can reflect bacteria is close to stationary phase.</li><li>All inducers (lactose &amp; arabinose) are taken into bacteria through active transport. To simplify simulation, we set this process as linear.</li></ol><h3 id="references-of-constants"><a class="anchor" href="#references-of-constants">#</a> References of Constants</h3><p>[1]: Stamatakis M, Mantzaris N V. <span class="exturl" data-url="aHR0cHM6Ly93d3cubmNiaS5ubG0ubmloLmdvdi9wdWJtZWQvMTkxODYxMjg=">Comparison of Deterministic and Stochastic Models of the lac Operon Genetic Network</span>[J]. Biophysical Journal, 2009, 96(3):887-906.</p><p>[2]: újvári, Andrea, Martin C T. Thermodynamic and Kinetic Measurements of Promoter Binding by T7 RNA Polymerase[J]. Biochemistry, 1996, 35(46):14574-14582.</p><p>[3] Lee J, Ramirez W F. Mathematical modeling of induced foreign protein production by recombinant bacteria[J]. Biotechnology and bioengineering, 1992, 39(6): 635-646.</p><p>[4] <span class="exturl" data-url="aHR0cHM6Ly8yMDE5LmlnZW0ub3JnL1RlYW06RnVkYW4tVFNJL01vZGVsIw==">https://2019.igem.org/Team:Fudan-TSI/Model#</span></p><h2 id="key-variables-in-model"><a class="anchor" href="#key-variables-in-model">#</a> Key Variables in Model</h2><p><img data-src="https://2021.igem.org/wiki/images/2/28/T--Fudan--modeling7.jpg" alt="pic"><br><img data-src="https://2021.igem.org/wiki/images/b/bd/T--Fudan--modeling8.jpg" alt="pic"><br><strong>Table 2.2 List of all the key variables in 5 categories: transcription factors, mRNAs, proteins, inducers, complexes. Concentration unit is mmol/mL.</strong></p><p>References of Variables</p><p>[5] Battesti A, Majdalani N, Gottesman S. The RpoS-mediated general stress response in Escherichia coli[J]. Annual review of microbiology, 2011, 65: 189-213.</p><p>[6] Tabib-Salazar A, Liu B, Barker D, et al. T7 phage factor required for managing RpoS in Escherichia coli[J]. Proceedings of the National Academy of Sciences, 2018, 115(23): E5353-E5362.</p><h2 id="key-processes-in-model"><a class="anchor" href="#key-processes-in-model">#</a> Key Processes in Model</h2><p>To clearly simulate our model, we set the processes below to explain our understanding of the whole reaction. All processes are presented with a kinetic model. Parameters on the left show <strong>the Substances Involved in the Reaction</strong>. Parameters on the arrow with a “K” are the kinetic constant present <strong>the Speed</strong> of this reaction. Parameters on the right are <strong>the Products</strong> of corresponding reaction. Thus, we can briefly introduce our model.</p><p><img data-src="https://2021.igem.org/wiki/images/5/53/T--Fudan--modeling9.jpg" alt="pic"><br><img data-src="https://2021.igem.org/wiki/images/c/ce/T--Fudan--modeling10.jpg" alt="pic"></p><p><img data-src="https://2021.igem.org/wiki/images/2/20/T--Fudan--modeling19.jpg" alt="pic"></p><p><img data-src="https://2021.igem.org/wiki/images/6/62/T--Fudan--modeling11.jpg" alt="pic"></p><p>**Table 2.3 List of all key processes divided (similar to parameters) into 8 categories. **</p><p><img data-src="https://2021.igem.org/wiki/images/5/53/T--Fudan--modeling21.jpg" alt="pic"><br><strong>Fig2.1. Describes our summary of all the biochemical processes involved, further simplified into 3 categories: state transitions, processes that facilitate transitions, and processes inhibiting transition. All reactions apart from degradation and dissociation are marked accordingly in this network.</strong></p><h2 id="key-differential-equations-in-model"><a class="anchor" href="#key-differential-equations-in-model">#</a> Key Differential Equations in Model</h2><p>We assumed all process as first-ordered reactions. To simulate the whole processes including transcription, translation, induction, degradation, inhibition, release. We next arranged the processes into differential equations of <strong>Time</strong> to solve our problem. The <strong>left</strong> shows the <strong>Rate of Concentration</strong> of corresponding parameter. The right shows all parameters that influence the rate.</p><p>Parameter itself represents its own concentration. To: 1. Choose the best plan for Igp5.7 expression, 2. Seek the best induction time. We here list some main parameters.</p><p><img data-src="https://2021.igem.org/wiki/images/2/2c/T--Fudan--modeling12.jpg" alt="pic"><br><strong>Table 2.4 List of all main parameters found in multiple equations.</strong></p><p>Due to the complexity of our model, it’s hard to list them all. Here is the link for all equations:</p><h1 id="parameter-tuning-and-validate"><a class="anchor" href="#parameter-tuning-and-validate">#</a> Parameter Tuning and Validate</h1><h2 id="simulate-with-runge-kutta-method"><a class="anchor" href="#simulate-with-runge-kutta-method">#</a> Simulate with Runge-Kutta Method</h2><p>Due to the complexity of the whole model, the analytical solution of function cannot be analyzed in elementary function. Floating points are essential in our model as we mentioned. Therefore, Euler's Method as a general simulation method cannot meet the accuracy. To match real curve to the most, we choose Runge-Kutta Method rather than Euler’s Method.</p><h2 id="parameter-tuning"><a class="anchor" href="#parameter-tuning">#</a> Parameter Tuning</h2><p>We assumed that the material in the model is sufficient to produce no matter inter or external products. Thus, to steady the model, the concentration of every parameter must be convergent. Fundamentally the convergence of mRNA is dependent on degradation kinetic constant and transcription kinetic constant while the initial concentration of transcription factors merely influences convergence value. In conclusion, mRNA degradation and transcription are tightly paired during adjustment. Degradation kinetic constants of certain proteins were found in articles [1], and thus <strong>transcription</strong>, <strong>translation</strong>, and <strong>degradation</strong> kinetic constants are determined.</p><p>The translation is adjusted the same way as transcription. Although it’s clear that there are several mechanisms to degrade protein due to molecular size, we here assumed that all protein involved shares the same degradation kinetic constant. RpoS (σs) levels are very low and increase as bacteria enter stationary phase. [3] We considered σs in our model. s70 is replaced by σs when bacteria get into stationary phase. (Figure 1)</p><p>The figure below is simulated with ode45 of Matlab. We can find initiation concentration of σ70 gradually goes down and finally convergents to 1.4. At the same time, the concentration of σs transcription factor goes up and reaches a steady phase with a concentration of 1.5. Endogenous protein produced by endogenous transcription factors, although some fluctuations were seen, can steady at 2.8. We thus complete our initiation and parameter tuning.</p><p><em>Endogenous Protein = P1 + P2</em></p><p><img data-src="https://2021.igem.org/wiki/images/1/18/T--Fudan--MODELSUP.jpg" alt="pic"><br><strong>Figure 3.1, Concentration of σ70 &amp; Mσ70, σs &amp; Mσs and Endogenous protein convergents within limited time</strong></p><p>σ70, free s70 transcription factor. σ70bind, bound s70 in transcription. Mσ70, mRNA of s70.</p><p>σs, free ss transcription factor. σsbind bound ss in transcription. Mσs, mRNA of σs.</p><p>Endogenous protein, protein expressed by both s70 &amp; σs.</p><h2 id="validation"><a class="anchor" href="#validation">#</a> Validation</h2><p>As we can find in the constant table, most kinetic constants are involving transcription, translation, and degradation. We examined the kinetic curve of our model with experiments. Although the absolute concentration is not the same as we get from the experiment, the result matches the experiments satisfyingly in total 6 parallel experiments after normalization. We also found that the overall expression time is not totally matched. Because the plasmid will not start expression immediately after transduction. Due to complexity, we did not simulate this period of time in the modeling. Therefore, we look for evidence from the real-time fluorescence data in other literature.[7] Generally, the experimental data can be seen within two hours under some experimental conditions.</p><p>With our assumptions, we find the kinetic constants of transcription, translation, and degradation paired. In this circumstance, the relationship between the parameters is determined and validated.</p><p><img data-src="https://2021.igem.org/wiki/images/0/07/T--Fudan--modeling16.jpg" alt="pic"><br><strong>Figure 3.2, Model protein matches well with experiment data after normalization and linear time correction</strong></p><p>P1, blue line, model protein.</p><p>GFP1~6 are experiment data, points</p><p><strong>Reference:</strong></p><p>[7]: Arnoldini M, Heck T, Blanco-Fernández A, et al. Monitoring of dynamic microbiological processes using real-time flow cytometry[J]. PloS one, 2013, 8(11): e80117.</p><h1 id="results"><a class="anchor" href="#results">#</a> Results</h1><h2 id="induction-addition"><a class="anchor" href="#induction-addition">#</a> Induction Addition</h2><p>Two states of inducers include intercellular and extracellular. We assumed active transport for inducer transmembrane. Set aside the quantity of internal ATP, we considered the kinetic modeling of transport as linear. Thus, the intercellular inducer concentration can be measured more effectively. Here we show a figure of lactose. It's clear that extracellular lactose is transported effectively while intercellular lactose is consumed indicating that the protein induced by lactose begins to express.</p><p><img data-src="https://2021.igem.org/wiki/images/0/0c/T--Fudan--modeling3.jpg" alt="pic"><br><strong>Figure 4.1, Transportof inducer</strong></p><p>lac. Inlac, intercellular lac.</p><p>Outlac, extracellular lac</p><h2 id="inhibition-of-transcription-factors"><a class="anchor" href="#inhibition-of-transcription-factors">#</a> Inhibition of Transcription Factors</h2><p>After initiation, we gave value to concentration of inducers. We can find an evident decrease of both transcription factors (blue lines) to one-third of the original concentration after inducer addition.</p><p><img data-src="https://2021.igem.org/wiki/images/e/ed/T--Fudan--modeling2.jpg" alt="pic"><br><strong>Figure 4.2, Concentration of σ70 &amp; Mσ70, σs &amp; Mσs decreases after inducer addition.</strong></p><p>σ70, free s70 transcription factor. σ70bind, bound s70 in transcription. Mσ70, mRNA of s70.</p><p>σs, free ss transcription factor. σsbind bound ss in transcription. Mσs, mRNA of σs.</p><h2 id="igp57-expression-simulation"><a class="anchor" href="#igp57-expression-simulation">#</a> Igp5.7 Expression Simulation</h2><p>Two ways of Igp5.7 expression are simulated with our established model. In the absence of inducer, we find that Igp5.7 without inducer will evidently inhibit σs and decrease the concentration while at the same time we can still find active transcription of σs mRNA. With low transcription factors, the endogenous protein will also decrease, suggesting a low growth rate of bacteria. Here we suppose that Igp5.7 without inducer would largely influence the growth rate of bacteria and finally reduce the efficiency of production.</p><p><img data-src="https://2021.igem.org/wiki/images/1/16/T--Fudan--modeling17.jpg" alt="pic"></p><p><img data-src="https://2021.igem.org/wiki/images/c/c0/T--Fudan--modeling.jpg" alt="pic"></p><p><img data-src="https://2021.igem.org/wiki/images/0/08/T--Fudan--modeling14.jpg" alt="pic"></p><p><img data-src="https://2021.igem.org/wiki/images/f/fd/T--Fudan--modelsup2.png" alt="pic"></p><p><strong>Figure 4.3 The plan that Igp5.7 with inducer shows better growth rate of bacteria than the other</strong></p><p>σ70, free s70 transcription factor. σ70bind, bound s70 in transcription. Mσ70, mRNA of s70.</p><p>σs, free ss transcription factor. σsbind bound ss in transcription. Mσs, mRNA of σs.</p><p>Endogenous protein, protein expressed by both s70 &amp; σs, can represent the growth rate of bacteria.</p><p>Igp5.7, the inhibitor. MIgp5.7, the mRNA of Igp5.7. σsIgp5.7, σs-Igp5.7 complex.</p><h2 id="harvest-target-protein-with-high-purity"><a class="anchor" href="#harvest-target-protein-with-high-purity">#</a> Harvest Target Protein with High Purity</h2><p>First, We measure the purity of our target protein with the equation:</p><p><img data-src="https://2021.igem.org/wiki/images/6/6e/T--Fudan--modeling18.jpg" alt="pic"><br>Apart from purity, the absolute concentration of GFP is also considered (Figure 5). We can find two normal distribution-shaped curves that represent the ratio and absolute concentration respectively. To steady the model, as we mentioned, we set degradation kinetic constant to convergent the terminal value while in experimental circumstance with limited material protein expressed may not degrade in such a rapid way. Thus, the range of harvest time may not satisfy real-time experiments. Whereas in the assumption that all protein shares the same degradation kinetic constant, parameter adjustment merely brings linear-like transform to our model, we can hold the point that the best harvest time may be in an interval rather than the longer the better.</p><p><img data-src="https://2021.igem.org/wiki/images/e/eb/T--Fudan--modeling15.jpg" alt="pic"></p><p><strong>Figure 4.4, Concentrations of GFP and Purity of GFP are normal distribution like curves</strong></p><p>The range marked is considered the proper time to harvest</p><p>Then, another important question we hope to answer is how the timespan between adding inducers lactose and arabinose could influence the purity of target protein. We set an additional condition: following lactose induction, arabinose is only transported through membrane after concentration of extracellular lactose has dropped below a certain threshold. As the concentration of extracellular lactose decreases linearly under our conditions, we can effectively simulate adding lactose and arabinose at separate times and adjust this timespan by setting different threshold values.</p><p>We derived a third-order function showing how target protein purity changes temporally when lactose and arabinose are added under different timespans.</p><p><img data-src="https://2021.igem.org/wiki/images/a/a0/T--Fudan--modeling5.jpg" alt="pic"><br><strong>Figure 4.4, Time-dependent purity of GFP curve with different inducer addition interval</strong></p><p>x-label: time unit: min</p><p>y-label:timespan</p><p>z-label: purity unit: 1</p><p>The most important findings from our results are that the purity-time curve varies very little along the timespan axis. This would indicate a counterintuitive conclusion: the timespan between adding inducers has little influence on the purity of target protein.</p><p><img data-src="https://2021.igem.org/wiki/images/6/6c/T--Fudan--modeling4.jpg" alt="pic"><br><strong>Figure 4.5, Time-dependent purity of GFP curve</strong></p><p>x-label: time unit: min</p><p>z-label: purity unit: 1</p><p>The result is more visually evident if we observe the purity-time quadrant and compare the curves. Significant differences can be found only over a long period of time (over 100 min), where a longer timespan between inducer additions leads to a slightly less decrease in purity following the peak.</p><p>It's important to note that the optimum harvesting time falls under 100 min, meaning to achieve maximum target protein purity, the harvesting time would be too low for timespan between inducer additions to make any difference.</p><p>This would mean that harvesting protein at the time interval we identified is sufficient for satisfactory yield and purity. There is one less major factor than we originally anticipated influencing Bst production, which is an important insight for our experiment and even future implementation.</p><h2 id="limitations"><a class="anchor" href="#limitations">#</a> Limitations</h2><p>Assumptions above can bring this model error. Some assumptions are unwritten but fundamental for protein expression. For example, physiologically inducer promoters can still bind to transcription factors and start to transcript [2]. Whereas, in our model, we gave tacit consent to the fact that protein would not leak without inducer addition. For general proteins, this numerical fluctuation would not influence the whole model. However, leakage of proteins like gp5.7 that are designed to inhibit internal protein expression would cause error.</p><p>Figure 6 below shows the relation between free gp5.7 and Total gp5.7 (<em>Free + Bound</em>). In our model, we can tell that although the total concentration of gp5.7 is considerable, free gp5.7 concentration is extremely low compared to total concentration due to high affinity and kinetic constant to bind transcription factors. Maybe only a little leakage can bring a considerable difference to transcription factor. But after all, the leakage is relatively neither hard to operate nor to detect for its low dose. In the end, we bring up the affinity and kinetic constant in transcription factor inhibition to compensate for the error brought by gp leakage.</p><p>A</p><p><img data-src="https://2021.igem.org/wiki/images/9/9d/T--Fudan--modeling20.jpg" alt="pic"></p><p>B</p><p><img data-src="https://2021.igem.org/wiki/images/5/5a/T--Fudan--description3.jpg" alt="pic"></p><p><strong>Figure 6. A: Comparison between Free gp5.7 and Total gp5.7. Concentration of total is far more than that of free</strong></p><p><strong>B: Concentration of free gp5.7</strong></p><p>Total = Free + Bound.</p><p>To summarize, we achieved an accurate description of the T7-E.coli expression model with a complex network of interrelated biochemical reactions. We included all processes (transcription, translation, inhibition, transmembrane transportation, etc.) in our ODEs and chose to run Runge-Kutta method with ode45 to achieve optimum fitting.</p><p></p><p>We derived all parameters reliably through references and rationale. We further tuned our parameters based on several important and reasonable assumptions：（i）We reasoned that the ideal curve for all variables must converge in a system with unlimited materials. This gave us a direct way to examine whether unknown parameters have been tuned correctly. (ii) We described bacteria metabolic stress with the rate of endogenous protein expression. (iii) We took minimum values into consideration to reflect how extremely small amounts of elements could participate in reactions.</p><p>Our model was validated with experimental data. In turn, we instructed the experiment by (i) finding the best strategy for composite parts; (ii) predetermining the optimum time interval for protein harvest; (iii) discovering that the timespan between inducer addition had little effect on the yield and purity of target protein.</p><p>Future teams can reference our validated host bacteria vector system model with other objectives, and learn from our insightful assumptions and rationale when tuning their own parameters.</p></div><footer><div class="meta"><span class="item"><span class="icon"><i class="ic i-calendar-check"></i> </span><span class="text">Edited on</span> <time title="Modified: 2021-10-22 07:25:44" itemprop="dateModified" datetime="2021-10-22T07:25:44+08:00">2021-10-22</time></span></div></footer></article></div><div class="wrap" id="comments"></div></div><div id="sidebar"><div class="inner"><div class="panels"><div class="inner"><div class="contents panel pjax" data-title="Contents"><ol class="toc"><li class="toc-item toc-level-1"><a class="toc-link" href="#key-problems"><span class="toc-number">1.</span> <span class="toc-text">Key Problems</span></a></li><li class="toc-item toc-level-1"><a class="toc-link" href="#key-parameters-and-processes"><span class="toc-number">2.</span> <span class="toc-text">Key Parameters and Processes</span></a><ol class="toc-child"><li class="toc-item toc-level-2"><a class="toc-link" href="#key-constants-in-model"><span class="toc-number">2.1.</span> <span class="toc-text">Key Constants in Model</span></a><ol class="toc-child"><li class="toc-item toc-level-3"><a class="toc-link" href="#assumptions"><span class="toc-number">2.1.1.</span> <span class="toc-text">Assumptions</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#references-of-constants"><span class="toc-number">2.1.2.</span> <span class="toc-text">References of Constants</span></a></li></ol></li><li class="toc-item toc-level-2"><a class="toc-link" href="#key-variables-in-model"><span class="toc-number">2.2.</span> <span class="toc-text">Key Variables in Model</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#key-processes-in-model"><span class="toc-number">2.3.</span> <span class="toc-text">Key Processes in Model</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#key-differential-equations-in-model"><span class="toc-number">2.4.</span> <span class="toc-text">Key Differential Equations in Model</span></a></li></ol></li><li class="toc-item toc-level-1"><a class="toc-link" href="#parameter-tuning-and-validate"><span class="toc-number">3.</span> <span class="toc-text">Parameter Tuning and Validate</span></a><ol class="toc-child"><li class="toc-item toc-level-2"><a class="toc-link" href="#simulate-with-runge-kutta-method"><span class="toc-number">3.1.</span> <span class="toc-text">Simulate with Runge-Kutta Method</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#parameter-tuning"><span class="toc-number">3.2.</span> <span class="toc-text">Parameter Tuning</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#validation"><span class="toc-number">3.3.</span> <span class="toc-text">Validation</span></a></li></ol></li><li class="toc-item toc-level-1"><a class="toc-link" href="#results"><span class="toc-number">4.</span> <span class="toc-text">Results</span></a><ol class="toc-child"><li class="toc-item toc-level-2"><a class="toc-link" href="#induction-addition"><span class="toc-number">4.1.</span> <span class="toc-text">Induction Addition</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#inhibition-of-transcription-factors"><span class="toc-number">4.2.</span> <span class="toc-text">Inhibition of Transcription Factors</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#igp57-expression-simulation"><span class="toc-number">4.3.</span> <span class="toc-text">Igp5.7 Expression Simulation</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#harvest-target-protein-with-high-purity"><span class="toc-number">4.4.</span> <span class="toc-text">Harvest Target Protein with High Purity</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#limitations"><span class="toc-number">4.5.</span> <span class="toc-text">Limitations</span></a></li></ol></li></ol></div><div class="related panel pjax" data-title="Related"></div><div class="overview panel" data-title="Overview"><div class="author" itemprop="author" itemscope itemtype="http://schema.org/Person"><img class="image" itemprop="image" alt="Tom Ghost Smith" data-src="/images/avatar.jpg"><p class="name" itemprop="name">Tom Ghost Smith</p><div class="description" itemprop="description">描述 123</div></div><nav class="state"><div class="item posts"><a href="/archives/"><span class="count">26</span> <span class="name">posts</span></a></div><div class="item categories"><a href="/categories/"><span class="count">8</span> <span class="name">categories</span></a></div><div class="item tags"><a href="/tags/"><span class="count">5</span> <span class="name">tags</span></a></div></nav><div class="social"><span class="exturl item github" data-url="aHR0cHM6Ly9naXRodWIuY29tL1RvbUdob3N0U21pdGg=" title="https:&#x2F;&#x2F;github.com&#x2F;TomGhostSmith"><i class="ic i-github"></i></span> <span class="exturl item twitter" data-url="aHR0cHM6Ly90d2l0dGVyLmNvbS95b3VybmFtZQ==" title="https:&#x2F;&#x2F;twitter.com&#x2F;yourname"><i class="ic i-twitter"></i></span> <span class="exturl item zhihu" data-url="aHR0cHM6Ly93d3cuemhpaHUuY29tL3Blb3BsZS95b3VybmFtZQ==" title="https:&#x2F;&#x2F;www.zhihu.com&#x2F;people&#x2F;yourname"><i class="ic i-zhihu"></i></span> <span class="exturl item music" data-url="aHR0cHM6Ly9tdXNpYy4xNjMuY29tLyMvdXNlci9ob21lP2lkPXlvdXJpZA==" title="https:&#x2F;&#x2F;music.163.com&#x2F;#&#x2F;user&#x2F;home?id&#x3D;yourid"><i class="ic i-cloud-music"></i></span> <span class="exturl item weibo" data-url="aHR0cHM6Ly93ZWliby5jb20veW91cm5hbWU=" title="https:&#x2F;&#x2F;weibo.com&#x2F;yourname"><i class="ic i-weibo"></i></span> <span class="exturl item about" data-url="aHR0cHM6Ly9hYm91dC5tZS95b3VybmFtZQ==" title="https:&#x2F;&#x2F;about.me&#x2F;yourname"><i class="ic i-address-card"></i></span></div><ul class="menu"><li class="item"><a href="/" rel="section"><i class="ic i-home"></i>Home</a></li><li class="item dropdown"><a href="javascript:void(0);"><i class="ic i-feather"></i>Posts</a><ul class="submenu"><li class="item"><a href="/archives/" rel="section"><i class="ic i-list-alt"></i>Archives</a></li><li class="item"><a href="/categories/" rel="section"><i class="ic i-th"></i>Categories</a></li><li class="item"><a href="/tags/" rel="section"><i class="ic i-tags"></i>Tags</a></li></ul></li><li class="item"><a href="/friends/" rel="section"><i class="ic i-heart"></i>Friends</a></li></ul></div></div></div><ul id="quick"><li class="prev pjax"></li><li class="up"><i class="ic i-arrow-up"></i></li><li class="down"><i class="ic i-arrow-down"></i></li><li class="next pjax"></li><li class="percent"></li></ul></div></div><div class="dimmer"></div></div></main><footer id="footer"><div class="inner"><div class="widgets"><div class="rpost pjax"><h2>Random Posts</h2><ul><li class="item"><div class="breadcrumb"><a href="/categories/computer-science/" title="In 计算机科学">计算机科学</a> <i class="ic i-angle-right"></i> <a href="/categories/computer-science/note/" title="In 二进制杂谈">二进制杂谈</a> <i class="ic i-angle-right"></i> <a href="/categories/computer-science/note/Theme-Shoka-Documentation/" title="In Theme Shoka Documentation">Theme Shoka Documentation</a></div><span><a href="/computer-science/note/theme-shoka-doc/config/" title="Step.2 基本配置">Step.2 基本配置</a></span></li><li class="item"><div class="breadcrumb"></div><span><a href="/images/" title="Images">Images</a></span></li><li class="item"><div class="breadcrumb"></div><span><a href="/long-title/" title="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam justo turpis, tincidunt ac convallis id.">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam justo turpis, tincidunt ac convallis id.</a></span></li><li class="item"><div class="breadcrumb"></div><span><a href="/tags/" title="Tags">Tags</a></span></li><li class="item"><div class="breadcrumb"></div><span><a href="/excerpts/" title="Excerpts">Excerpts</a></span></li><li class="item"><div class="breadcrumb"><a href="/categories/computer-science/" title="In 计算机科学">计算机科学</a> <i class="ic i-angle-right"></i> <a href="/categories/computer-science/note/" title="In 二进制杂谈">二进制杂谈</a> <i class="ic i-angle-right"></i> <a href="/categories/computer-science/note/Theme-Shoka-Documentation/" title="In Theme Shoka Documentation">Theme Shoka Documentation</a></div><span><a href="/computer-science/note/theme-shoka-doc/dependents/" title="Step.1 依赖插件">Step.1 依赖插件</a></span></li><li class="item"><div class="breadcrumb"><a href="/categories/Foo/" title="In Foo">Foo</a> <i class="ic i-angle-right"></i> <a href="/categories/Foo/Bar/" title="In Bar">Bar</a> <i class="ic i-angle-right"></i> <a href="/categories/Foo/Bar/Baz/" title="In Baz">Baz</a></div><span><a href="/categories/" title="Categories">Categories</a></span></li><li class="item"><div class="breadcrumb"></div><span><span class="exturl" data-url="aHR0cDovL3d3dy5nb29nbGUuY29tLw==" title="Link Post">Link Post<i class="ic i-link-alt"></i></span></span></li><li class="item"><div class="breadcrumb"></div><span><span class="exturl" data-url="aHR0cDovL3d3dy5nb29nbGUuY29tLw==" title="www.google.com">www.google.com<i class="ic i-link-alt"></i></span></span></li><li class="item"><div class="breadcrumb"></div><span><a href="/%E6%97%A5%E6%9C%AC%E8%AA%9E%E3%83%86%E3%82%B9%E3%83%88/" title="日本語テスト">日本語テスト</a></span></li></ul></div><div><h2>Recent Comments</h2><ul class="leancloud-recent-comment"></ul></div></div><div class="status"><div class="copyright">&copy; 2010 – <span itemprop="copyrightYear">2021</span> <span class="with-love"><i class="ic i-sakura rotate"></i> </span><span class="author" itemprop="copyrightHolder">Tom Ghost Smith @ Yume Shoka</span></div><div class="count"><span class="post-meta-item-icon"><i class="ic i-chart-area"></i> </span><span title="Symbols count total">NaNm words</span> <span class="post-meta-divider">|</span> <span class="post-meta-item-icon"><i class="ic i-coffee"></i> </span><span title="Reading time total">NaN:aN</span></div><div class="powered-by">Powered by <span class="exturl" data-url="aHR0cHM6Ly9oZXhvLmlv">Hexo</span> & Theme.<span class="exturl" data-url="aHR0cHM6Ly9naXRodWIuY29tL2FtZWhpbWUvaGV4by10aGVtZS1zaG9rYQ==">Shoka</span></div></div></div></footer></div><script data-config type="text/javascript">var LOCAL={path:"wiki/Modeling.html",favicon:{show:"（●´3｀●）Goooood",hide:"(´Д｀)Booooom"},search:{placeholder:"Search for Posts",empty:"We didn't find any results for the search: ${query}",stats:"${hits} results found in ${time} ms"},valine:!0,fancybox:!0,copyright:'Copied to clipboard successfully! <br> All articles in this blog are licensed under <i class="ic i-creative-commons"></i>BY-NC-SA.',ignores:[function(e){return e.includes("#")},function(e){return new RegExp(LOCAL.path+"$").test(e)}]}</script><script src="https://cdn.polyfill.io/v2/polyfill.js"></script><script src="//cdn.jsdelivr.net/combine/npm/pace-js@1.0.2/pace.min.js,npm/pjax@0.2.8/pjax.min.js,npm/whatwg-fetch@3.4.0/dist/fetch.umd.min.js,npm/animejs@3.2.0/lib/anime.min.js,npm/algoliasearch@4/dist/algoliasearch-lite.umd.js,npm/instantsearch.js@4/dist/instantsearch.production.min.js,npm/lozad@1/dist/lozad.min.js,npm/quicklink@2/dist/quicklink.umd.js"></script><script src="/js/app.js?v=0.2.5"></script></body></html>